__NUXT_JSONP__("/topics", (function(a,b,c,d,e,f,g,h,i,j,k,l,m,n,o){return {data:[{markdownFiles:[{slug:"s01-genome-browsers",title:"Genome Browsers",img:"genome-browsers.png",caption:"Some different types of genomic tracks visualized in the [UCSC Genome Browser](https:\u002F\u002Fgenome.ucsc.edu\u002F)",ingress:"Genomic tracks are made for genome browsers",toc:[],body:{type:m,children:[{type:b,tag:j,props:{},children:[{type:b,tag:k,props:{},children:[{type:a,value:"Genomic track files"}]},{type:a,value:" were originally designed and optimised to be displayed within "},{type:b,tag:k,props:{},children:[{type:a,value:"genome browsers"}]},{type:a,value:". A genome browser\nis a type of genome-wide visualisation software able to display various datasets as parallel tracks along the DNA\nsequence coordinates of typically reference genomes. A multitude of genome browsers are available as online web services\non top of a database of track files and\u002For as installable software. A selection genome browsers are listed below:"}]},{type:a,value:c},{type:b,tag:"ui-markdown-table",props:{":headers":"['Genome Browser', 'Web service', 'Scientific domain', 'Installable software', 'Prog. language', 'Library']",":items":"[\n['[Ensembl](https:\u002F\u002Fwww.ensembl.org\u002F)', 'Yes', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002FEnsembl)', 'Perl \u002F Javascript', 'No'],\n['[UCSC Genome Browser](https:\u002F\u002Fgenome.ucsc.edu\u002F)',   'Yes', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002FucscGenomeBrowser\u002Fkent)', 'C \u002F HTML', 'No'],\n['[UCSC Xena](http:\u002F\u002Fxena.ucsc.edu\u002F)', 'Yes', 'Cancer multi-omics', '[Yes](https:\u002F\u002Fgithub.com\u002FucscXena)', 'Clojure \u002F Javascript', 'No'],\n['[WashU Epigenome Browser](https:\u002F\u002Fepigenomegateway.wustl.edu\u002F)', 'Yes', 'Epigenomics', '[Yes](https:\u002F\u002Fgithub.com\u002Ftwlab\u002Feg-react)', 'Javascript', 'No'],\n['[Zenbu \u002F FANTOM](https:\u002F\u002Ffantom.gsc.riken.jp\u002Fzenbu\u002F)','Yes', 'Gene regulation', '[Yes](https:\u002F\u002Fsourceforge.net\u002Fprojects\u002Fzenbu\u002F)', 'C++ \u002F Perl \u002F Javascript', 'No'],\n['[GTEx Locus Browser](https:\u002F\u002Fgtexportal.org\u002Fhome\u002FlocusBrowserPage\u002FACTN3)', 'Yes', 'Human variation (gene-centric)', '[Only demo](https:\u002F\u002Fgithub.com\u002Fbroadinstitute\u002Fgtex-viz\u002Ftree\u002Fmaster\u002FlocusBrowser)', 'Javascript', 'No'],\n['[DECIPHER](https:\u002F\u002Fwww.deciphergenomics.org\u002F)', 'Yes', 'Clinical data', 'No', '', ''],\n['[GTEx IGV Browser](https:\u002F\u002Fgtexportal.org\u002Fhome\u002FbrowseEqtls)', 'Yes', 'Human variation', 'No', '', ''],\n['[Plant Epigenome Browser](https:\u002F\u002Fepigenome.genetics.uga.edu\u002FPlantEpigenome\u002F)', 'Yes', 'Plant epigenomics', 'No', '', ''],\n['[Genoverse](https:\u002F\u002Fgenoverse.org\u002F)', 'No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002Fwtsi-web\u002FGenoverse)', 'Javascript', 'Yes'],\n['[GIVE](https:\u002F\u002Fzhong-lab-ucsd.github.io\u002FGIVE_homepage\u002F)', 'No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002FZhong-Lab-UCSD\u002FGenomic-Interactive-Visualization-Engine)', 'HTML5 \u002F Javascript', 'Yes'],\n['[HiGlass](https:\u002F\u002Fhiglass.io\u002F)', 'No', 'Chromatin interaction', '[Yes](https:\u002F\u002Fgithub.com\u002Fhiglass)', 'Python \u002F Javascript', 'No'],\n['[IGV](https:\u002F\u002Figv.org\u002F)','No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002Figvteam)', 'Java \u002F Javascript', 'Yes'],\n['[JBrowse](https:\u002F\u002Fjbrowse.org\u002Fjb2\u002F)', 'No', 'General', '[Yes](https:\u002F\u002Fjbrowse.org\u002Fjb2\u002Fdownload\u002F)', 'Javascript', 'Yes'],\n['[NGB](https:\u002F\u002Flifescience.opensource.epam.com\u002Fngb\u002Findex.html)','No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002Fepam\u002FNGB)', 'Java \u002F Javascript', 'No'],\n['[pyGenomeTracks](https:\u002F\u002Fpygenometracks.readthedocs.io\u002Fen\u002Flatest\u002F)','No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002Fdeeptools\u002FpyGenomeTracks)', 'Python', 'Yes'],\n['[Web Apollo](http:\u002F\u002Fgenomearchitect.org\u002F)', 'No', 'Collaborative genome annotation', '[Yes](https:\u002F\u002Fgithub.com\u002FGMOD\u002FApollo)', 'Groovy \u002F Java \u002F Javascript', 'No'],\n]",":table-number":"1",":table-caption":"'Various genome browsers deployed as web services with possibly domain-restricted track ' +\n'databases and\u002For available as installable software.'"},children:[{type:a,value:"\n\n\n"}]},{type:a,value:c},{type:b,tag:j,props:{},children:[{type:a,value:"For more complete lists of genome browsers, please visit "},{type:b,tag:e,props:{href:"https:\u002F\u002Fbio.tools\u002Ft?page=1&q=%27Genome%20browser%27&sort=citationDate&ord=desc",rel:[f,g,h],target:i},children:[{type:a,value:"bio.tools"}]},{type:a,value:"\nor the "},{type:b,tag:e,props:{href:"https:\u002F\u002Fcmdcolin.github.io\u002Fawesome-genome-visualization",rel:[f,g,h],target:i},children:[{type:a,value:"awesome genome visualization"}]},{type:a,value:" web page (genome browser\nsoftware only)."}]}]},dir:n,path:"\u002Ftopics\u002Fs01-genome-browsers",extension:o,createdAt:l,updatedAt:l},{slug:"s02-genomic-tracks",title:"Genomic Tracks",img:"genomic-tracks.png",caption:"Geometrical delineation of genomic tracks into 15 track types [(Gundersen, S *et al.*, 2011)](https:\u002F\u002Fdoi.org\u002F10.1186\u002F1471-2105-12-494)",ingress:"Tracks annotate the DNA with condensed data",toc:[],body:{type:m,children:[{type:b,tag:j,props:{},children:[{type:b,tag:k,props:{},children:[{type:a,value:"Genomic tracks"}]},{type:a,value:" refer to data files that annotate DNA reference sequence positions and can be visualized in genome\nbrowsers. Track files represent summaries of the raw data according to specific criteria and granularity.\nFor example: “hot spot” regions (with a high number of reads), values deviating from expectations, or cross-genomic\nlinks representing closeness in 3D. In essence, the condensed data in track files relate to the raw data much like an\nabstract describes a scientific publication. This data reduction allows researchers to scan large amounts of data to\ndefine a hypothesis before carrying out more accurate analyses.\n"},{type:b,tag:"ui-quote-text",props:{":quote":"\"A genomic track relates to the raw data much like an abstract describes a scientific publication.\""},children:[{type:a,value:c}]},{type:a,value:"\nMany types of data from genomic analyses can be represented as genomic tracks, for example:"}]},{type:a,value:c},{type:b,tag:"ul",props:{},children:[{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Gene regions, repeating elements, conserved regions"}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Chromatin accessibility (e.g., DNase I Hypersensitivity)"}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Binding of Transcription Factors (TFs) to DNA"}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Histone modifications along the DNA"}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Gene expression, Gene fusions, Transcription Start Sites (TSS)"}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Cis-regulatory elements (promoters, enhancers, ...)"}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"DNA methylation"}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"3D chromatin interaction"}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Single Nucleotide Polymorphisms (SNPs) from Genome-Wide Association Studies (GWAS)"}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Single Nucleotide Variants (SNVs) and Copy Number Variants (CNVs) in cancer cells"}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Virus insertion sites"}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"plus many more"}]},{type:a,value:c}]},{type:a,value:c},{type:b,tag:j,props:{},children:[{type:a,value:"Gradually, genomic tracks have also become a "},{type:b,tag:k,props:{},children:[{type:a,value:"de facto"}]},{type:a,value:" standard to store, distribute and analyse genome-wide datasets,\nmainly because of their efficient compression and indexing utilities. Many bioinformatics analyses are now being\ndistributed, either privately or publicly, using related file formats. Tracks are still predominantly used for graphical\ndisplay, but can also be queried by statistical analysis tools, such as\nthe "},{type:b,tag:e,props:{href:"https:\u002F\u002Fhyperbrowser.uio.no\u002F",rel:[f,g,h],target:i},children:[{type:a,value:"GSuite HyperBrowser"}]},{type:a,value:" , "},{type:b,tag:e,props:{href:"https:\u002F\u002Fblueprint-data.bsc.es\u002F",rel:[f,g,h],target:i},children:[{type:a,value:"EPICO"}]},{type:a,value:",\nor "},{type:b,tag:e,props:{href:"https:\u002F\u002Fdeepblue.mpi-inf.mpg.de\u002F",rel:[f,g,h],target:i},children:[{type:a,value:"DeepBlue"}]},{type:a,value:"."}]}]},dir:n,path:"\u002Ftopics\u002Fs02-genomic-tracks",extension:o,createdAt:l,updatedAt:l}]}],fetch:{},mutations:[]}}("text","element","\n","li","a","nofollow","noopener","noreferrer","_blank","p","em","2022-01-07T14:38:55.351Z","root","\u002Ftopics",".md")));