__NUXT_JSONP__("/topics", (function(a,b,c,d,e,f,g,h,i,j,k,l,m,n,o,p,q,r,s,t,u,v,w,x,y,z,A,B,C,D,E,F){return {data:[{markdownFiles:[{slug:"s01-genomic-tracks",title:"Genomic Tracks",img:"genomic-tracks.jpeg",caption:"Unwinding of a DNA molecule from its compact chromatin form. Each nucleotide pair represents a coordinate position which can be annotated with genomic track data. (Image&nbsp;is&nbsp;[©&nbsp;scienceDISPLAY&nbsp;–&nbsp;stock.adobe.com](https:\u002F\u002Fstock.adobe.com\u002Fimages\u002Fdna-nucleosome-chromosom\u002F42940588))",ingress:"Tracks annotate the DNA with condensed data",toc:[],body:{type:p,children:[{type:b,tag:f,props:{},children:[{type:b,tag:e,props:{},children:[{type:a,value:u}]},{type:a,value:" refer to data files that annotate DNA reference sequence positions and can be visualized in genome\nbrowsers. Track files represent summaries of the raw data according to specific criteria and granularity.\nFor example: “hot spot” regions (with a high number of reads), values deviating from expectations, or cross-genomic\nlinks representing closeness in 3D. In essence, the condensed data in track files relate to the raw data much like an\nabstract describes a scientific publication. This data reduction allows researchers to scan large amounts of data to\ndefine a hypothesis before carrying out more accurate analyses.\n"},{type:b,tag:v,props:{":quote":"\"A genomic track relates to the raw data much like an abstract describes a scientific publication.\""},children:[{type:a,value:c}]},{type:a,value:"\nMany types of data from genomic analyses can be represented as genomic tracks, for example:"}]},{type:a,value:c},{type:b,tag:"ul",props:{},children:[{type:a,value:c},{type:b,tag:g,props:{},children:[{type:a,value:"Gene regions, repeating elements, conserved regions"}]},{type:a,value:c},{type:b,tag:g,props:{},children:[{type:a,value:"Chromatin accessibility (e.g., DNase I Hypersensitivity)"}]},{type:a,value:c},{type:b,tag:g,props:{},children:[{type:a,value:"Binding of Transcription Factors (TFs) to DNA"}]},{type:a,value:c},{type:b,tag:g,props:{},children:[{type:a,value:"Histone modifications along the DNA"}]},{type:a,value:c},{type:b,tag:g,props:{},children:[{type:a,value:"Gene expression, Gene fusions, Transcription Start Sites (TSS)"}]},{type:a,value:c},{type:b,tag:g,props:{},children:[{type:a,value:"Cis-regulatory elements (promoters, enhancers, ...)"}]},{type:a,value:c},{type:b,tag:g,props:{},children:[{type:a,value:"DNA methylation"}]},{type:a,value:c},{type:b,tag:g,props:{},children:[{type:a,value:"3D chromatin interaction"}]},{type:a,value:c},{type:b,tag:g,props:{},children:[{type:a,value:"Single Nucleotide Polymorphisms (SNPs) from Genome-Wide Association Studies (GWAS)"}]},{type:a,value:c},{type:b,tag:g,props:{},children:[{type:a,value:"Single Nucleotide Variants (SNVs) and Copy Number Variants (CNVs) in cancer cells"}]},{type:a,value:c},{type:b,tag:g,props:{},children:[{type:a,value:"Virus insertion sites"}]},{type:a,value:c},{type:b,tag:g,props:{},children:[{type:a,value:"plus many more"}]},{type:a,value:c}]},{type:a,value:c},{type:b,tag:f,props:{},children:[{type:a,value:"Gradually, genomic tracks have also become a "},{type:b,tag:e,props:{},children:[{type:a,value:"de facto"}]},{type:a,value:" standard to store, distribute and analyse genome-wide datasets,\nmainly because of their efficient compression and indexing utilities. Many bioinformatics analyses are now being\ndistributed, either privately or publicly, using related file formats. Tracks are still predominantly used for graphical\ndisplay, but can also be queried by statistical analysis tools, such as\nthe "},{type:b,tag:h,props:{href:"https:\u002F\u002Fhyperbrowser.uio.no\u002F",rel:[i,j,k],target:l},children:[{type:a,value:"GSuite HyperBrowser"}]},{type:a,value:" , "},{type:b,tag:h,props:{href:"https:\u002F\u002Fblueprint-data.bsc.es\u002F",rel:[i,j,k],target:l},children:[{type:a,value:"EPICO"}]},{type:a,value:",\nor "},{type:b,tag:h,props:{href:"https:\u002F\u002Fdeepblue.mpi-inf.mpg.de\u002F",rel:[i,j,k],target:l},children:[{type:a,value:"DeepBlue"}]},{type:a,value:n}]}]},dir:q,path:"\u002Ftopics\u002Fs01-genomic-tracks",extension:r,createdAt:m,updatedAt:m},{slug:"s02-genome-browsers",title:"Genome Browsers",img:"genome-browsers.png",caption:"Some different types of genomic tracks visualized in the [UCSC Genome Browser](https:\u002F\u002Fgenome.ucsc.edu\u002F)",ingress:"Genomic tracks are made for genome browsers",toc:[],body:{type:p,children:[{type:b,tag:f,props:{},children:[{type:b,tag:e,props:{},children:[{type:a,value:"Genomic track files"}]},{type:a,value:" were originally designed and optimized to be displayed within "},{type:b,tag:e,props:{},children:[{type:a,value:"genome browsers"}]},{type:a,value:". A genome browser\nis a type of genome-wide visualization software able to display various datasets as parallel tracks along the DNA\nsequence coordinates of typically reference genomes. A multitude of genome browsers are available as online web services\non top of a database of track files and\u002For as installable software. A selection genome browsers are listed below:"}]},{type:a,value:c},{type:b,tag:"ui-markdown-table",props:{":headers":"['Genome Browser', 'Web service', 'Scientific domain', 'Installable software', 'Prog. language', 'Library']",":items":"[\n['[Ensembl](https:\u002F\u002Fwww.ensembl.org\u002F)', 'Yes', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002FEnsembl)', 'Perl \u002F Javascript', 'No'],\n['[UCSC Genome Browser](https:\u002F\u002Fgenome.ucsc.edu\u002F)',   'Yes', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002FucscGenomeBrowser\u002Fkent)', 'C \u002F HTML', 'No'],\n['[UCSC Xena](http:\u002F\u002Fxena.ucsc.edu\u002F)', 'Yes', 'Cancer multi-omics', '[Yes](https:\u002F\u002Fgithub.com\u002FucscXena)', 'Clojure \u002F Javascript', 'No'],\n['[WashU Epigenome Browser](https:\u002F\u002Fepigenomegateway.wustl.edu\u002F)', 'Yes', 'Epigenomics', '[Yes](https:\u002F\u002Fgithub.com\u002Ftwlab\u002Feg-react)', 'Javascript', 'No'],\n['[Zenbu \u002F FANTOM](https:\u002F\u002Ffantom.gsc.riken.jp\u002Fzenbu\u002F)','Yes', 'Gene regulation', '[Yes](https:\u002F\u002Fsourceforge.net\u002Fprojects\u002Fzenbu\u002F)', 'C++ \u002F Perl \u002F Javascript', 'No'],\n['[GTEx Locus Browser](https:\u002F\u002Fgtexportal.org\u002Fhome\u002FlocusBrowserPage\u002FACTN3)', 'Yes', 'Human variation (gene-centric)', '[Only demo](https:\u002F\u002Fgithub.com\u002Fbroadinstitute\u002Fgtex-viz\u002Ftree\u002Fmaster\u002FlocusBrowser)', 'Javascript', 'No'],\n['[DECIPHER](https:\u002F\u002Fwww.deciphergenomics.org\u002F)', 'Yes', 'Clinical data', 'No', '', ''],\n['[GTEx IGV Browser](https:\u002F\u002Fgtexportal.org\u002Fhome\u002FbrowseEqtls)', 'Yes', 'Human variation', 'No', '', ''],\n['[Plant Epigenome Browser](https:\u002F\u002Fepigenome.genetics.uga.edu\u002FPlantEpigenome\u002F)', 'Yes', 'Plant epigenomics', 'No', '', ''],\n['[Genoverse](https:\u002F\u002Fgenoverse.org\u002F)', 'No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002Fwtsi-web\u002FGenoverse)', 'Javascript', 'Yes'],\n['[GIVE](https:\u002F\u002Fzhong-lab-ucsd.github.io\u002FGIVE_homepage\u002F)', 'No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002FZhong-Lab-UCSD\u002FGenomic-Interactive-Visualization-Engine)', 'HTML5 \u002F Javascript', 'Yes'],\n['[HiGlass](https:\u002F\u002Fhiglass.io\u002F)', 'No', 'Chromatin interaction', '[Yes](https:\u002F\u002Fgithub.com\u002Fhiglass)', 'Python \u002F Javascript', 'No'],\n['[IGV](https:\u002F\u002Figv.org\u002F)','No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002Figvteam)', 'Java \u002F Javascript', 'Yes'],\n['[JBrowse](https:\u002F\u002Fjbrowse.org\u002Fjb2\u002F)', 'No', 'General', '[Yes](https:\u002F\u002Fjbrowse.org\u002Fjb2\u002Fdownload\u002F)', 'Javascript', 'Yes'],\n['[NGB](https:\u002F\u002Flifescience.opensource.epam.com\u002Fngb\u002Findex.html)','No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002Fepam\u002FNGB)', 'Java \u002F Javascript', 'No'],\n['[pyGenomeTracks](https:\u002F\u002Fpygenometracks.readthedocs.io\u002Fen\u002Flatest\u002F)','No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002Fdeeptools\u002FpyGenomeTracks)', 'Python', 'Yes'],\n['[Web Apollo](http:\u002F\u002Fgenomearchitect.org\u002F)', 'No', 'Collaborative genome annotation', '[Yes](https:\u002F\u002Fgithub.com\u002FGMOD\u002FApollo)', 'Groovy \u002F Java \u002F Javascript', 'No'],\n]",":table-number":w,":table-caption":"'Various genome browsers deployed as web services with possibly domain-restricted track ' +\n'databases and\u002For available as installable software.'"},children:[{type:a,value:x}]},{type:a,value:c},{type:b,tag:f,props:{},children:[{type:a,value:"For more complete lists of genome browsers, please visit "},{type:b,tag:h,props:{href:"https:\u002F\u002Fbio.tools\u002Ft?page=1&q=%27Genome%20browser%27&sort=citationDate&ord=desc",rel:[i,j,k],target:l},children:[{type:a,value:"bio.tools"}]},{type:a,value:"\nor the "},{type:b,tag:h,props:{href:"https:\u002F\u002Fcmdcolin.github.io\u002Fawesome-genome-visualization",rel:[i,j,k],target:l},children:[{type:a,value:"awesome genome visualization"}]},{type:a,value:" web page (genome browser\nsoftware only)."}]},{type:a,value:c},{type:b,tag:v,props:{":quote":"\"The FAIRtracks team advocate the use of the term \\\"track\\\" also in analysis scenarios outside the realm of genome visualisation. \""},children:[{type:a,value:x}]},{type:a,value:c},{type:b,tag:f,props:{},children:[{type:a,value:"The term \"track\" implies a mental visual model of elements located along some line. In the domain of genomics\nthis typically relates to a coordinate system defined by a reference genome sequence collection.\nThis is a crucial model to keep in mind also for non-visual analysis scenarios. There exists, to our knowledge, no\nother term in broad use which is both general and precise enough to cover the same heterogeneity of data\ncontent, data representation and analysis scenarios as the term \"track\" (or a variation\nthereof). Hence, the FAIRtracks team advocate the use of the term \"track\" also in analysis scenarios\noutside the realm of genome visualization."}]}]},dir:q,path:"\u002Ftopics\u002Fs02-genome-browsers",extension:r,createdAt:m,updatedAt:m},{slug:"s03-track-types",title:"Track Types",img:"track-types.png",caption:"Geometrical delineation of genomic tracks into 15 track types [(Gundersen, S *et al.*, 2011)](https:\u002F\u002Fdoi.org\u002F10.1186\u002F1471-2105-12-494)",ingress:"Track data can be categorized into 15 theoretical track types",toc:[],body:{type:p,children:[{type:b,tag:f,props:{},children:[{type:a,value:"As exemplified in the section "},{type:b,tag:h,props:{href:"#s02-genomic-tracks"},children:[{type:a,value:u}]},{type:a,value:", a multitude of different types of genome-wide\ndata, experimental and otherwise, can be condensed in the form of track files. The variety of data that can be\nexpressed in track form has consequences for the representation of the data, both for mathematical modeling\nand for the file formats used for storage and exchange."}]},{type:a,value:c},{type:b,tag:f,props:{},children:[{type:a,value:"Gundersen, S "},{type:b,tag:e,props:{},children:[{type:a,value:y}]},{type:a,value:z},{type:b,tag:h,props:{href:A,rel:[i,j,k],target:l},children:[{type:a,value:B}]},{type:a,value:" reviewed and categorized the landscape of\ntrack data representation. This paper proposed a unified conceptual framework which delineated the\nmodel space into 15 possible mathematical\u002Fgeometrical models, named "},{type:b,tag:d,props:{},children:[{type:a,value:C}]},{type:a,value:". These 15 models can be used to\nrepresent, visualize, and analyze fundamentally different types of track data."}]},{type:a,value:c},{type:b,tag:f,props:{},children:[{type:a,value:"Logically, these "},{type:b,tag:d,props:{},children:[{type:a,value:C}]},{type:a,value:" correspond to whether four different core properties of the data are present:\n"},{type:b,tag:d,props:{},children:[{type:a,value:s}]},{type:a,value:D},{type:b,tag:d,props:{},children:[{type:a,value:o}]},{type:a,value:D},{type:b,tag:d,props:{},children:[{type:a,value:E}]},{type:a,value:", and "},{type:b,tag:d,props:{},children:[{type:a,value:t}]},{type:a,value:". Figure 3 illustrates the similarities and\ndifferences of these 15\ndifferent types of track data. If we look closely at the fifteen combinations, starting at the top left of Figure 3\nand moving downwards, an interesting pattern appears. From Gundersen, S "},{type:b,tag:e,props:{},children:[{type:a,value:y}]},{type:a,value:",\n"},{type:b,tag:h,props:{href:A,rel:[i,j,k],target:l},children:[{type:a,value:B}]},{type:a,value:z},{type:b,tag:"sup",props:{id:"fnref-1"},children:[{type:b,tag:h,props:{href:"#fn-1",className:["footnote-ref"]},children:[{type:a,value:w}]}]},{type:a,value:":"}]},{type:a,value:c},{type:b,tag:"blockquote",props:{},children:[{type:a,value:c},{type:b,tag:f,props:{},children:[{type:a,value:"We start at the base case where the only core informational property is the "},{type:b,tag:d,props:{},children:[{type:a,value:s}]},{type:a,value:" between the track elements. In\nthis case, each track element represents an exact base pair location on the genome, denoting "},{type:b,tag:e,props:{},children:[{type:a,value:"e.g. viral insertion\nsites"}]},{type:a,value:". We call this track type "},{type:b,tag:d,props:{},children:[{type:a,value:"Points (P)"}]},{type:a,value:". Adding informative values to this case, "},{type:b,tag:e,props:{},children:[{type:a,value:"e.g. associating SNPs with\nallele frequencies"}]},{type:a,value:", we get the track type "},{type:b,tag:d,props:{},children:[{type:a,value:"Valued Points (VP)"}]},{type:a,value:". In\nthe next two cases, the property "},{type:b,tag:d,props:{},children:[{type:a,value:o}]},{type:a,value:" is added, resulting in the track types "},{type:b,tag:d,props:{},children:[{type:a,value:"Segments (S)"}]},{type:a,value:F},{type:b,tag:d,props:{},children:[{type:a,value:"Valued\nSegments (VS)"}]},{type:a,value:". Segments are probably the most common track type of existing tracks, representing "},{type:b,tag:e,props:{},children:[{type:a,value:"e.g. genes or\nexons, TF binding sites, or DNA methylation regions"}]},{type:a,value:". Valued segments could denote "},{type:b,tag:e,props:{},children:[{type:a,value:"e.g. genes with\nassociated expression levels"}]},{type:a,value:n}]},{type:a,value:c},{type:b,tag:f,props:{},children:[{type:a,value:"Moving on, we remove the "},{type:b,tag:d,props:{},children:[{type:a,value:E}]},{type:a,value:F},{type:b,tag:d,props:{},children:[{type:a,value:s}]},{type:a,value:" properties, leaving only "},{type:b,tag:d,props:{},children:[{type:a,value:o}]},{type:a,value:". Such tracks consist of\nsegments covering all base pairs of the genome, i.e. a partition of the genome into potentially unequal pieces.\nHence, the track type is called "},{type:b,tag:d,props:{},children:[{type:a,value:"Genome Partition (GP)"}]},{type:a,value:". Examples of this track type include the "},{type:b,tag:e,props:{},children:[{type:a,value:"partition\nof a genome into chromosomes or predicted chromatin states"}]},{type:a,value:". Adding a "},{type:b,tag:d,props:{},children:[{type:a,value:"value"}]},{type:a,value:" to each part of a partition creates a\nStep Function (SF), covering the whole genome with values. Examples of such tracks include "},{type:b,tag:e,props:{},children:[{type:a,value:"mapped sequence read\ncounts (e.g. a BAM file), or the signal files that are typically generated in a ChIP-seq workflow"}]},{type:a,value:". Removing the\n"},{type:b,tag:d,props:{},children:[{type:a,value:o}]},{type:a,value:" core property, the step function track is transformed into a track of type Function (F), where every\nbase pair has an associated value. Examples of function tracks are tracks with close dependency on the genome\nsequence, such as "},{type:b,tag:e,props:{},children:[{type:a,value:"GC content tracks, or the actual genome sequence itself"}]},{type:a,value:". We call the seven track types outlined\nhere for "},{type:b,tag:d,props:{},children:[{type:a,value:"basic track types"}]},{type:a,value:n}]},{type:a,value:c},{type:b,tag:f,props:{},children:[{type:a,value:"The fourth core informational property, "},{type:b,tag:d,props:{},children:[{type:a,value:t}]},{type:a,value:", can be envisioned as an orthogonal extension to the\nprevious discussion. Adding interconnections, or edges, to the seven track types previously outlined (first column\nof Figure 3) defines linked versions of the same track types, e.g. "},{type:b,tag:d,props:{},children:[{type:a,value:"Linked Segments (LS)"}]},{type:a,value:" or "},{type:b,tag:d,props:{},children:[{type:a,value:"Linked Step Function\n(LSF)"}]},{type:a,value:" (second column of Figure 3). Examples of linked track dataset include "},{type:b,tag:e,props:{},children:[{type:a,value:"ChIA-PET chromatin interactions\n("},{type:b,tag:d,props:{},children:[{type:a,value:"Linked Segments, LS"}]},{type:a,value:") and Hi-C contact maps ("},{type:b,tag:d,props:{},children:[{type:a,value:"Linked Genome Partition, LGP"}]},{type:a,value:")"}]},{type:a,value:n}]},{type:a,value:c},{type:b,tag:f,props:{},children:[{type:a,value:"[...]"}]},{type:a,value:c},{type:b,tag:f,props:{},children:[{type:a,value:"To complete the picture, a last track type needs to be defined. If only the "},{type:b,tag:d,props:{},children:[{type:a,value:t}]},{type:a,value:" core property is\ndefined, track elements do not have gaps between them, lengths, or values. [...] The linked\nbase pairs track type is mostly suggestive at this point. [...] We refer to the eight linked track types as the\n"},{type:b,tag:d,props:{},children:[{type:a,value:"extended track types"}]},{type:a,value:n}]},{type:a,value:c}]},{type:a,value:c},{type:b,tag:f,props:{},children:[{type:a,value:"The track type concept is useful not only for categorizing tracked data itself but also for a\ncategorizing possible analyses in terms of the track types of their input and output files. For example, "},{type:b,tag:e,props:{},children:[{type:a,value:"ChIP-seq peak\ncalling"}]},{type:a,value:" can be viewed as a data reduction operation, transforming input track files of type "},{type:b,tag:d,props:{},children:[{type:a,value:"Step Function (SF)"}]},{type:a,value:"\n("},{type:b,tag:e,props:{},children:[{type:a,value:"i.e. sequence read files"}]},{type:a,value:") into output track files of type "},{type:b,tag:d,props:{},children:[{type:a,value:"Valued Segments (VS)"}]},{type:a,value:", where the values represent the\nintensities of the peaks that are called."}]},{type:a,value:c},{type:b,tag:f,props:{},children:[{type:a,value:"Nota that all 15 track types are supported by the FAIRTracks metadata standard, as well as the GTrack and GSuite file\nformats, which are all described in the "},{type:b,tag:"nuxt-link",props:{to:"\u002Fstandards"},children:[{type:a,value:"Standards"}]},{type:a,value:" page."}]},{type:a,value:c},{type:b,tag:"div",props:{className:["footnotes"]},children:[{type:a,value:c},{type:b,tag:"hr",props:{},children:[]},{type:a,value:c},{type:b,tag:"ol",props:{},children:[{type:a,value:c},{type:b,tag:g,props:{id:"fn-1"},children:[{type:a,value:"The quote is slightly edited. Text formatting have been added and some examples have been substituted by more recent ones."},{type:b,tag:h,props:{href:"#fnref-1",className:["footnote-backref"]},children:[{type:a,value:"↩"}]}]},{type:a,value:c}]},{type:a,value:c}]}]},dir:q,path:"\u002Ftopics\u002Fs03-track-types",extension:r,createdAt:m,updatedAt:m}]}],fetch:{},mutations:[]}}("text","element","\n","strong","em","p","li","a","nofollow","noopener","noreferrer","_blank","2022-01-11T14:58:41.170Z",".","lengths","root","\u002Ftopics",".md","gaps","interconnections","Genomic tracks","ui-quote-text","1","\n\n\n","et al."," ","https:\u002F\u002Fdoi.org\u002F10.1186\u002F1471-2105-12-494","(2011)","track types",", ","values"," and ")));