__NUXT_JSONP__("/topics", (function(a,b,c,d,e,f,g,h,i,j,k,l,m,n,o,p,q,r,s,t,u,v,w,x,y,z,A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,Q,R,S,T,U,V,W,X,Y,Z,_,$,aa,ab,ac,ad,ae,af,ag,ah,ai,aj,ak,al,am,an,ao,ap,aq,ar,as,at,au,av,aw,ax,ay,az,aA,aB,aC,aD,aE,aF,aG,aH,aI,aJ,aK,aL,aM,aN,aO,aP,aQ,aR,aS,aT,aU,aV,aW,aX,aY,aZ,a_,a$,ba,bb,bc){return {data:[{markdownFiles:[{slug:"s01-genomic-tracks",title:"Genomic Tracks",img:T,caption:"Unwinding of a DNA molecule from its compact chromatin form. Each nucleotide pair represents a coordinate position which can be annotated with genomic track data. (Image&nbsp;is&nbsp;[©&nbsp;scienceDISPLAY&nbsp;–&nbsp;stock.adobe.com](https:\u002F\u002Fstock.adobe.com\u002Fimages\u002Fdna-nucleosome-chromosom\u002F42940588))",ingress:"Tracks annotate the DNA with condensed data",toc:[],body:{type:r,children:[{type:b,tag:d,props:{},children:[{type:b,tag:m,props:{},children:[{type:a,value:U}]},{type:a,value:" refer to data files that annotate DNA reference sequence positions and can be visualized in genome\nbrowsers. Track files represent summaries of the raw data according to specific criteria and granularity.\nFor example: “hot spot” regions (with a high number of reads), values deviating from expectations, or cross-genomic\nlinks representing closeness in 3D. In essence, the condensed data in track files relate to the raw data much like an\nabstract describes a scientific publication. This data reduction allows researchers to scan large amounts of data to\ndefine a hypothesis before carrying out more accurate analyses.\n"},{type:b,tag:V,props:{":quote":"\"A genomic track relates to the raw data much like an abstract describes a scientific publication.\""},children:[{type:a,value:c}]},{type:a,value:"\nMany types of data from genomic analyses can be represented as genomic tracks, for example:"}]},{type:a,value:c},{type:b,tag:"ul",props:{},children:[{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Gene regions, repeating elements, conserved regions"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Chromatin accessibility (e.g., DNase I Hypersensitivity)"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Binding of Transcription Factors (TFs) to DNA"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Histone modifications along the DNA"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Gene expression, Gene fusions, Transcription Start Sites (TSS)"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Cis-regulatory elements (promoters, enhancers, ...)"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"DNA methylation"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"3D chromatin interaction"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Single Nucleotide Polymorphisms (SNPs) from Genome-Wide Association Studies (GWAS)"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Single Nucleotide Variants (SNVs) and Copy Number Variants (CNVs) in cancer cells"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Virus insertion sites"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"plus many more"}]},{type:a,value:c}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Gradually, genomic tracks have also become a "},{type:b,tag:m,props:{},children:[{type:a,value:"de facto"}]},{type:a,value:" standard to store, distribute and analyze genome-wide datasets,\nmainly because of their efficient compression and indexing utilities. Many bioinformatics analyses are now being\ndistributed, either privately or publicly, using related file formats. Tracks are still predominantly used for graphical\ndisplay, but can also be queried by statistical analysis tools, such as\nthe "},{type:b,tag:e,props:{href:"https:\u002F\u002Fhyperbrowser.uio.no\u002F",rel:[g,h,i],target:j},children:[{type:a,value:"GSuite HyperBrowser"}]},{type:a,value:" , "},{type:b,tag:e,props:{href:"https:\u002F\u002Fblueprint-data.bsc.es\u002F",rel:[g,h,i],target:j},children:[{type:a,value:"EPICO"}]},{type:a,value:",\nor "},{type:b,tag:e,props:{href:"https:\u002F\u002Fdeepblue.mpi-inf.mpg.de\u002F",rel:[g,h,i],target:j},children:[{type:a,value:"DeepBlue"}]},{type:a,value:w}]}]},dir:s,path:"\u002Ftopics\u002Fs01-genomic-tracks",extension:t,createdAt:q,updatedAt:q},{slug:"s02-genome-browsers",title:"Genome Browsers",img:W,caption:"Some different types of genomic tracks visualized in the [UCSC Genome Browser](https:\u002F\u002Fgenome.ucsc.edu\u002F)",ingress:"Genomic tracks are made for genome browsers",toc:[],body:{type:r,children:[{type:b,tag:d,props:{},children:[{type:b,tag:m,props:{},children:[{type:a,value:"Genomic track files"}]},{type:a,value:" were originally designed and optimized to be displayed within "},{type:b,tag:m,props:{},children:[{type:a,value:"genome browsers"}]},{type:a,value:". A genome browser\nis a type of genome-wide visualization software able to display various datasets as parallel tracks along the DNA\nsequence coordinates of typically reference genomes. A multitude of genome browsers are available as online web services\non top of a database of track files and\u002For as installable software. A selection genome browsers are listed below:"}]},{type:a,value:c},{type:b,tag:X,props:{":headers":"['Genome Browser', 'Web service', 'Scientific domain', 'Installable software', 'Prog. language', 'Library']",":items":"[\n['[Ensembl](https:\u002F\u002Fwww.ensembl.org\u002F)', 'Yes', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002FEnsembl)', 'Perl \u002F Javascript', 'No'],\n['[UCSC Genome Browser](https:\u002F\u002Fgenome.ucsc.edu\u002F)',   'Yes', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002FucscGenomeBrowser\u002Fkent)', 'C \u002F HTML', 'No'],\n['[UCSC Xena](http:\u002F\u002Fxena.ucsc.edu\u002F)', 'Yes', 'Cancer multi-omics', '[Yes](https:\u002F\u002Fgithub.com\u002FucscXena)', 'Clojure \u002F Javascript', 'No'],\n['[WashU Epigenome Browser](https:\u002F\u002Fepigenomegateway.wustl.edu\u002F)', 'Yes', 'Epigenomics', '[Yes](https:\u002F\u002Fgithub.com\u002Ftwlab\u002Feg-react)', 'Javascript', 'No'],\n['[Zenbu \u002F FANTOM](https:\u002F\u002Ffantom.gsc.riken.jp\u002Fzenbu\u002F)','Yes', 'Gene regulation', '[Yes](https:\u002F\u002Fsourceforge.net\u002Fprojects\u002Fzenbu\u002F)', 'C++ \u002F Perl \u002F Javascript', 'No'],\n['[GTEx Locus Browser](https:\u002F\u002Fgtexportal.org\u002Fhome\u002FlocusBrowserPage\u002FACTN3)', 'Yes', 'Human variation (gene-centric)', '[Only demo](https:\u002F\u002Fgithub.com\u002Fbroadinstitute\u002Fgtex-viz\u002Ftree\u002Fmaster\u002FlocusBrowser)', 'Javascript', 'No'],\n['[DECIPHER](https:\u002F\u002Fwww.deciphergenomics.org\u002F)', 'Yes', 'Clinical data', 'No', '', ''],\n['[GTEx IGV Browser](https:\u002F\u002Fgtexportal.org\u002Fhome\u002FbrowseEqtls)', 'Yes', 'Human variation', 'No', '', ''],\n['[Plant Epigenome Browser](https:\u002F\u002Fepigenome.genetics.uga.edu\u002FPlantEpigenome\u002F)', 'Yes', 'Plant epigenomics', 'No', '', ''],\n['[Genoverse](https:\u002F\u002Fgenoverse.org\u002F)', 'No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002Fwtsi-web\u002FGenoverse)', 'Javascript', 'Yes'],\n['[GIVE](https:\u002F\u002Fzhong-lab-ucsd.github.io\u002FGIVE_homepage\u002F)', 'No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002FZhong-Lab-UCSD\u002FGenomic-Interactive-Visualization-Engine)', 'HTML5 \u002F Javascript', 'Yes'],\n['[HiGlass](https:\u002F\u002Fhiglass.io\u002F)', 'No', 'Chromatin interaction', '[Yes](https:\u002F\u002Fgithub.com\u002Fhiglass)', 'Python \u002F Javascript', 'No'],\n['[IGV](https:\u002F\u002Figv.org\u002F)','No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002Figvteam)', 'Java \u002F Javascript', 'Yes'],\n['[JBrowse](https:\u002F\u002Fjbrowse.org\u002Fjb2\u002F)', 'No', 'General', '[Yes](https:\u002F\u002Fjbrowse.org\u002Fjb2\u002Fdownload\u002F)', 'Javascript', 'Yes'],\n['[NGB](https:\u002F\u002Flifescience.opensource.epam.com\u002Fngb\u002Findex.html)','No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002Fepam\u002FNGB)', 'Java \u002F Javascript', 'No'],\n['[pyGenomeTracks](https:\u002F\u002Fpygenometracks.readthedocs.io\u002Fen\u002Flatest\u002F)','No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002Fdeeptools\u002FpyGenomeTracks)', 'Python', 'Yes'],\n['[Web Apollo](http:\u002F\u002Fgenomearchitect.org\u002F)', 'No', 'Collaborative genome annotation', '[Yes](https:\u002F\u002Fgithub.com\u002FGMOD\u002FApollo)', 'Groovy \u002F Java \u002F Javascript', 'No'],\n]",":table-number":O,":table-caption":Y},children:[{type:a,value:P}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"For more complete lists of genome browsers, please visit "},{type:b,tag:e,props:{href:"https:\u002F\u002Fbio.tools\u002Ft?page=1&q=%27Genome%20browser%27&sort=citationDate&ord=desc",rel:[g,h,i],target:j},children:[{type:a,value:"bio.tools"}]},{type:a,value:"\nor the "},{type:b,tag:e,props:{href:"https:\u002F\u002Fcmdcolin.github.io\u002Fawesome-genome-visualization",rel:[g,h,i],target:j},children:[{type:a,value:"awesome genome visualization"}]},{type:a,value:" web page (genome browser\nsoftware only)."}]},{type:a,value:c},{type:b,tag:V,props:{":quote":"\"The FAIRtracks team advocate the use of the term \\\"track\\\" also in analysis scenarios outside the realm of genome visualisation. \""},children:[{type:a,value:P}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The term \"track\" implies a mental visual model of elements located along some line. In the domain of genomics\nthis typically relates to a coordinate system defined by a reference genome sequence collection.\nThis is a crucial model to keep in mind also for non-visual analysis scenarios. There exists, to our knowledge, no\nother term in broad use which is both general and precise enough to cover the same heterogeneity of data\ncontent, data representation and analysis scenarios as the term \"track\" (or a variation\nthereof). Hence, the FAIRtracks team advocate the use of the term \"track\" also in analysis scenarios\noutside the realm of genome visualization."}]}]},dir:s,path:"\u002Ftopics\u002Fs02-genome-browsers",extension:t,createdAt:q,updatedAt:q},{slug:"s03-track-types",title:"Track Types",img:Z,caption:"Geometrical delineation of genomic tracks into 15 track types [(Gundersen, S *et al.*, 2011)](https:\u002F\u002Fdoi.org\u002F10.1186\u002F1471-2105-12-494)",ingress:"Track data can be categorized into 15 theoretical track types",toc:[],body:{type:r,children:[{type:b,tag:d,props:{},children:[{type:a,value:"As exemplified in the section "},{type:b,tag:e,props:{href:"#s02-genomic-tracks"},children:[{type:a,value:U}]},{type:a,value:", a multitude of different types of genome-wide\ndata, experimental and otherwise, can be condensed in the form of track files. The variety of data that can be\nexpressed in track form has consequences for the representation of the data, both for mathematical modeling\nand for the file formats used for storage and exchange."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Gundersen, S "},{type:b,tag:m,props:{},children:[{type:a,value:_}]},{type:a,value:$},{type:b,tag:e,props:{href:aa,rel:[g,h,i],target:j},children:[{type:a,value:ab}]},{type:a,value:" reviewed and categorized the landscape of\ntrack data representation. This paper proposed a unified conceptual framework which delineated the\nmodel space into 15 possible mathematical\u002Fgeometrical models, named "},{type:b,tag:f,props:{},children:[{type:a,value:ac}]},{type:a,value:". These 15 models can be used to\nrepresent, visualize, and analyze fundamentally different types of track data."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Logically, these "},{type:b,tag:f,props:{},children:[{type:a,value:ac}]},{type:a,value:" correspond to whether four different core properties of the data are present:\n"},{type:b,tag:f,props:{},children:[{type:a,value:Q}]},{type:a,value:ad},{type:b,tag:f,props:{},children:[{type:a,value:D}]},{type:a,value:ad},{type:b,tag:f,props:{},children:[{type:a,value:ae}]},{type:a,value:", and "},{type:b,tag:f,props:{},children:[{type:a,value:R}]},{type:a,value:". Figure 3 illustrates the similarities and\ndifferences of these 15\ndifferent types of track data. If we look closely at the fifteen combinations, starting at the top left of Figure 3\nand moving downwards, an interesting pattern appears. From Gundersen, S "},{type:b,tag:m,props:{},children:[{type:a,value:_}]},{type:a,value:",\n"},{type:b,tag:e,props:{href:aa,rel:[g,h,i],target:j},children:[{type:a,value:ab}]},{type:a,value:$},{type:b,tag:"sup",props:{id:"fnref-1"},children:[{type:b,tag:e,props:{href:"#fn-1",className:["footnote-ref"]},children:[{type:a,value:O}]}]},{type:a,value:":"}]},{type:a,value:c},{type:b,tag:"blockquote",props:{},children:[{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"We start at the base case where the only core informational property is the "},{type:b,tag:f,props:{},children:[{type:a,value:Q}]},{type:a,value:" between the track elements. In\nthis case, each track element represents an exact base pair location on the genome, denoting "},{type:b,tag:m,props:{},children:[{type:a,value:"e.g. viral insertion\nsites"}]},{type:a,value:". We call this track type "},{type:b,tag:f,props:{},children:[{type:a,value:"Points (P)"}]},{type:a,value:". Adding informative values to this case, "},{type:b,tag:m,props:{},children:[{type:a,value:"e.g. associating SNPs with\nallele frequencies"}]},{type:a,value:", we get the track type "},{type:b,tag:f,props:{},children:[{type:a,value:"Valued Points (VP)"}]},{type:a,value:". In\nthe next two cases, the property "},{type:b,tag:f,props:{},children:[{type:a,value:D}]},{type:a,value:" is added, resulting in the track types "},{type:b,tag:f,props:{},children:[{type:a,value:"Segments (S)"}]},{type:a,value:E},{type:b,tag:f,props:{},children:[{type:a,value:"Valued\nSegments (VS)"}]},{type:a,value:". Segments are probably the most common track type of existing tracks, representing "},{type:b,tag:m,props:{},children:[{type:a,value:"e.g. genes or\nexons, TF binding sites, or DNA methylation regions"}]},{type:a,value:". Valued segments could denote "},{type:b,tag:m,props:{},children:[{type:a,value:"e.g. genes with\nassociated expression levels"}]},{type:a,value:w}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Moving on, we remove the "},{type:b,tag:f,props:{},children:[{type:a,value:ae}]},{type:a,value:E},{type:b,tag:f,props:{},children:[{type:a,value:Q}]},{type:a,value:" properties, leaving only "},{type:b,tag:f,props:{},children:[{type:a,value:D}]},{type:a,value:". Such tracks consist of\nsegments covering all base pairs of the genome, i.e. a partition of the genome into potentially unequal pieces.\nHence, the track type is called "},{type:b,tag:f,props:{},children:[{type:a,value:"Genome Partition (GP)"}]},{type:a,value:". Examples of this track type include the "},{type:b,tag:m,props:{},children:[{type:a,value:"partition\nof a genome into chromosomes or predicted chromatin states"}]},{type:a,value:". Adding a "},{type:b,tag:f,props:{},children:[{type:a,value:"value"}]},{type:a,value:" to each part of a partition creates a\nStep Function (SF), covering the whole genome with values. Examples of such tracks include "},{type:b,tag:m,props:{},children:[{type:a,value:"mapped sequence read\ncounts (e.g. a BAM file), or the signal files that are typically generated in a ChIP-seq workflow"}]},{type:a,value:". Removing the\n"},{type:b,tag:f,props:{},children:[{type:a,value:D}]},{type:a,value:" core property, the step function track is transformed into a track of type Function (F), where every\nbase pair has an associated value. Examples of function tracks are tracks with close dependency on the genome\nsequence, such as "},{type:b,tag:m,props:{},children:[{type:a,value:"GC content tracks, or the actual genome sequence itself"}]},{type:a,value:". We call the seven track types outlined\nhere for "},{type:b,tag:f,props:{},children:[{type:a,value:"basic track types"}]},{type:a,value:w}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The fourth core informational property, "},{type:b,tag:f,props:{},children:[{type:a,value:R}]},{type:a,value:", can be envisioned as an orthogonal extension to the\nprevious discussion. Adding interconnections, or edges, to the seven track types previously outlined (first column\nof Figure 3) defines linked versions of the same track types, e.g. "},{type:b,tag:f,props:{},children:[{type:a,value:"Linked Segments (LS)"}]},{type:a,value:" or "},{type:b,tag:f,props:{},children:[{type:a,value:"Linked Step Function\n(LSF)"}]},{type:a,value:" (second column of Figure 3). Examples of linked track dataset include "},{type:b,tag:m,props:{},children:[{type:a,value:"ChIA-PET chromatin interactions\n("},{type:b,tag:f,props:{},children:[{type:a,value:"Linked Segments, LS"}]},{type:a,value:") and Hi-C contact maps ("},{type:b,tag:f,props:{},children:[{type:a,value:"Linked Genome Partition, LGP"}]},{type:a,value:")"}]},{type:a,value:w}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"[...]"}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"To complete the picture, a last track type needs to be defined. If only the "},{type:b,tag:f,props:{},children:[{type:a,value:R}]},{type:a,value:" core property is\ndefined, track elements do not have gaps between them, lengths, or values. [...] The linked\nbase pairs track type is mostly suggestive at this point. [...] We refer to the eight linked track types as the\n"},{type:b,tag:f,props:{},children:[{type:a,value:"extended track types"}]},{type:a,value:w}]},{type:a,value:c}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The track type concept is useful not only for categorizing tracked data itself but also for a\ncategorizing possible analyses in terms of the track types of their input and output files. For example, "},{type:b,tag:m,props:{},children:[{type:a,value:"ChIP-seq peak\ncalling"}]},{type:a,value:" can be viewed as a data reduction operation, transforming input track files of type "},{type:b,tag:f,props:{},children:[{type:a,value:"Step Function (SF)"}]},{type:a,value:"\n("},{type:b,tag:m,props:{},children:[{type:a,value:"i.e. sequence read files"}]},{type:a,value:") into output track files of type "},{type:b,tag:f,props:{},children:[{type:a,value:"Valued Segments (VS)"}]},{type:a,value:", where the values represent the\nintensities of the peaks that are called."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Note that all 15 track types are supported by the FAIRTracks metadata standard, as well as the GTrack and GSuite file\nformats, which are all described in the "},{type:b,tag:S,props:{to:"\u002Fstandards"},children:[{type:a,value:"Standards"}]},{type:a,value:" page."}]},{type:a,value:c},{type:b,tag:"div",props:{className:["footnotes"]},children:[{type:a,value:c},{type:b,tag:"hr",props:{},children:[]},{type:a,value:c},{type:b,tag:"ol",props:{},children:[{type:a,value:c},{type:b,tag:p,props:{id:"fn-1"},children:[{type:a,value:"The quote is slightly edited. Text formatting have been added and some examples have been substituted by more recent ones."},{type:b,tag:e,props:{href:"#fnref-1",className:["footnote-backref"]},children:[{type:a,value:"↩"}]}]},{type:a,value:c}]},{type:a,value:c}]}]},dir:s,path:"\u002Ftopics\u002Fs03-track-types",extension:t,createdAt:q,updatedAt:q},{slug:"s04-finding-tracks",title:"Finding Tracks",img:x,caption:l,ingress:l,toc:[],body:{type:r,children:[{type:b,tag:d,props:{},children:[{type:a,value:"The FAIRtracks project maintains a draft standard for metadata about genomic track datasets. The aim is that the\nstandard will function as a minimal metadata exchange standard for harmonization of\nheterogeneous track metadata available in various [track collections or data portals, as described below]\n(\u002Fs05-track-collections). Once a process has been set up to transform existing or novel track collection to follow\nthe FAIRtracks standard (which in most cases also includes aspects of [FAIRification]\n(\u002Fs07-fairification)), the harmonized metadata can be imported into a central service for discovery of track files\nacross the various track collections that are integrated, named "},{type:b,tag:S,props:{to:"\u002Fservices#trackfind"},children:[{type:a,value:"TrackFind"}]},{type:a,value:w}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"TrackFind provides both a web user interface as well a REST API to allow access by downstream tools and scripts.\nIn the web portal, TrackFind provides a hierarchical browser of the various metadata fields mapped to the standard\nincluding summary lists of the different values that are represented in the indexed metadata. Through the graphical user\ninterface, these values can be used to define categorical search queries (in the form of a SQL query). The results\nof the queries are available for browsing or downloadable as JSON or GSuite formats."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The following screencast demonstrates use of track find to locate set of track files from the BLUEPRINT track\ncollection:"}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The FAIRtracks metadata standard is a minimal standard, it is designed to be interoperable with other resources and\nserviced. One example of such interoperability is the ability to launch external websites describing particular\nvocabulary terms or records of source material in other repositories. For more about this, see sections on\n"},{type:b,tag:S,props:{to:"\u002Fs08-identifiers"},children:[{type:a,value:af}]},{type:a,value:E},{type:b,tag:e,props:{href:ag},children:[{type:a,value:ah}]},{type:a,value:w}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Integrating with TrackFind. A key feature of TrackFind is the ability to integrates search functionality in other\nsoftware frameworks and tools. Currently this integration can be implemented by making use of the rest API,\ndocumented here. We aim to develop libraries in common premium language is that rap this API to simplify use an\nintegration,. Any contributions to this and is highly appreciated in. Currently, we have integrated TrackFind with\nthe GSuite HyperBrowser and EPICO.\nThe track find client tool in the dynamic have a browser illustrates a powerful feature of the TrackFind API: when\nspecifying the meta-data filled the criteria, the user will only be presented values that are available given the\nprevious filters that has been selected. Hence, the user will be guided interactively at every step in the\nfiltering process, reducing the probability for empty search results. For example, the user can start off by\nselecting transcription facto chipset As the experiment type, alongside a set of Target transcription factors, and\nwill then be presented with a list of Sample types (cell types, cell lines, etc.), where the TF‘s question I have\nbeen Investigated. This interactive reduction in selectable values can be extended to any metadata fields, e.g.,\nprojects information, file formats, Data usage limitations."}]}]},dir:s,path:"\u002Ftopics\u002Fs04-finding-tracks",extension:t,createdAt:q,updatedAt:q},{slug:"s05-track-collections",title:"Track Collections",img:ai,caption:"Most recommanded standard recommended by policies [(fairsharing.org)](https:\u002F\u002Ffairsharing.org\u002Fsummary-statistics)",ingress:"Access genomic tracks using Track collection",toc:[],body:{type:r,children:[{type:b,tag:d,props:{},children:[{type:a,value:"Track collections are structures that group one or more types of resources such as databases, standards or policies.\nThey can represent a domain, a project or an organization. Deposition of data relies on multiple standards for\ntracks deposit; consequently, available data and metadata may differ from collection to collection."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Many efforts have been made to uniformize these standards to follow the FAIR data principles. For example, the\nMinimum Information About a Microarray Experiment (MIAME) standard has been adopted by the most popular track\ncollections such as "},{type:b,tag:e,props:{href:"https:\u002F\u002Fwww.ebi.ac.uk\u002Farrayexpress\u002F",rel:[g,h,i],target:j},children:[{type:a,value:"ArrayExpress"}]},{type:a,value:" or [Gene Expression Omnibus]("},{type:b,tag:e,props:{href:aj,rel:[g,h,i],target:j},children:[{type:a,value:aj}]},{type:a,value:".\nncbi.nlm.nih.gov\u002Fgeo\u002F) (GEO). Another example is regarding Novel DNA sequence, novel RNA sequence, and novel genome\nassembly data that must be deposited to repositories that are part of the International Nucleotide Sequence\nCollaboration (INSDC) following a standard feature table. Consequently, tracks can be featured on multiple\ncollections with or without a cross identifier."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Projects exist to help find a relevant collection to one's interest such as [Repository Finder]\n("},{type:b,tag:e,props:{href:ak,rel:[g,h,i],target:j},children:[{type:a,value:ak}]},{type:a,value:") and "},{type:b,tag:e,props:{href:"https:\u002F\u002Ffairsharing.org",rel:[g,h,i],target:j},children:[{type:a,value:"FAIRsharing.org"}]},{type:a,value:". Moreover, Track hubs are\ncreated by researchers or projects to easily visualize genomic tracks in popular genome browsers such as [UCSC] or\nEnsembl and can be found on [UCSC Track hub list]("},{type:b,tag:e,props:{href:al,rel:[g,h,i],target:j},children:[{type:a,value:al}]},{type:a,value:"? redirect=manual&source=genome.ucsc.edu)\nor "},{type:b,tag:e,props:{href:"https:\u002F\u002Fwww.trackhubregistry.org",rel:[g,h,i],target:j},children:[{type:a,value:"Track Hub registtry"}]},{type:a,value:w}]},{type:a,value:c},{type:b,tag:X,props:{":headers":"['Database name', 'Scientific domain', 'Organism(s)', 'Assay type(s)', 'Organisation', 'Notes']",":items":"[\n['[Array Express](https:\u002F\u002Fwww.ebi.ac.uk\u002Farrayexpress\u002F)','Generic data warehouse','Multiple','Multiple','EMBL-EBI',''],\n['[Gene Expression Omnibus (GEO)](http:\u002F\u002Fwww.ncbi.nlm.nih.gov\u002Fgeo\u002F)','Generic data warehouse','Multiple','Multiple','National Center for Biotechnology Information',''],\n['[ENCODE](https:\u002F\u002Fwww.encodeproject.org)','Epigenomics','H. sapiens, M. musculus','DNA methylation \u002F histone modification \u002F chromatin accessibility \u002F transcription factor binding \u002F RNA-binding proteins \u002F  mRNA profiling \u002F ncRNA expression profiling \u002F 3D chromatin structure \u002F DNA replication timing','ENCODE consortium','Contain Roadmap and ModENCODE data, data in Array express, GEO and IHEC'],\n['[Roadmap Epigenomics](http:\u002F\u002Fwww.roadmapepigenomics.org)','Epigenomics \u002F Stem cells \u002F primary ex vivo tissues','H. sapiens','DNA methylation \u002F histone modifications \u002F chromatin accessibility \u002F small RNA transcripts','Roadmap Epigenomics Mapping Consortium (NIH)',''],\n['[ModENCODE](http:\u002F\u002Fwww.modencode.org)','Epigenomics \u002F Genomics','C. elegans, D. melanogaster','gene structure \u002F mRNA profiling \u002F ncRNA expression profiling \u002F transcription factor binding \u002F histone modifications \u002F histone replacement \u002F chromatin structure \u002F DNA replication initiation \u002F DNA replication timing \u002F copy number variation','ModEncode Consortium  (NIH)',''],\n['[Blueprint](https:\u002F\u002Fwww.blueprint-epigenome.eu)','Epigenomics \u002F Haematopoietic cells','H. sapiens, M. musculus','DNA methylation \u002F transcription factor binding \u002F chromatin accessibility \u002F mRNA profiling','BLUEPRINT Consortium  (IHEC)','Data in EGA'],\n['[FANTOM](https:\u002F\u002Ffantom.gsc.riken.jp)','Epigenomics','H. sapiens, M. musculus','mRNA profiling \u002F ncRNA expression profiling','Functional ANnoTation Of the Mammalian genome Consortium',''],\n['[International Human Epigenome Consortium (IHEC)](https:\u002F\u002Fepigenomesportal.ca)','Epigenomics','H. sapiens, M. musculus','DNA methylation \u002F histone modifications \u002F mRNA profiling','International Human Epigenome Consortium (IHEC)','Contain Blueprint, ENCODE data'],\n['[Cancer Genome Atlas Program (TCGA)](https:\u002F\u002Fportal.gdc.cancer.gov)','Epigenomics \u002F Genomics \u002F Cancer','H. sapiens','clinical data \u002F copy number variation \u002F single nucleotide variation \u002F DNA sequence variation \u002F imaging \u002F DNA methylation \u002F miRNA expression profiling \u002F mRNA expression profiling \u002F protein expression profiling','NCI (NIH) and the National Human Genome Research','Portal with CPTAC, MMRF, CMI, TARGET, HCMI and CGCI data'],\n['[International Cancer Genome Consortium (ICGC)](https:\u002F\u002Fdcc.icgc.org)','Epigenomics \u002F Genomics \u002F Cancer','H. sapiens','clinical data \u002F DNA methylation \u002F miRNA expression profiling \u002F mRNA expression profiling \u002F DNA sequence variation \u002F copy number variation','NCI (NIH) and the International Cancer Genome Consortium','Contain PCAWG data'],\n['[Pan-Cancer Analysis of Whole Genomes (PCAWG)](https:\u002F\u002Fdcc.icgc.org\u002Fpcawg\u002F)','Genomics \u002F Cancer','H. sapiens','clinical data \u002F copy number variation \u002F single nucleotide variation \u002F DNA sequence variation','Pan-Cancer Analysis of Whole Genomes consortium',''],\n['[DECIPHER](https:\u002F\u002Fwww.deciphergenomics.org)','Genomics \u002F Rare disease','H. sapiens','clinical data \u002F copy number variation \u002F DNA sequence variation','Sanger Institute',''],\n['[GTEx Portal](https:\u002F\u002Fgtexportal.org)','Epigenomics \u002F Genomics','H. sapiens','clinical data \u002F snRNA expression profiling \u002F mRNA expression profiling \u002F DNA sequence variation \u002F single nucleotide variation \u002F eQTL','The Genotype-Tissue Expression consortium',''],\n['[GWAS Catalog](https:\u002F\u002Fwww.ebi.ac.uk\u002Fgwas\u002F)','Genomics','H. sapiens','single nucleotide variation','National Human Genome Research Institute (NIH) and European Bioinformatics Institute',''],\n['[RNACentral](https:\u002F\u002Frnacentral.org)','Epigenomics','H. sapiens, M. musculus, A. thaliana, C. elegans, D. rerio, D. melanogaster. S. pombe','ncRNA expression profiling','European Bioinformatics Institute','Integration access of multiple databases \\*'],\n['[MirBase](https:\u002F\u002Fwww.mirbase.org)','Epigenomics','Multiple (Alveolata, Metazoa, Mycetozoa, Viridiplantae, Viruses)','miRNA expression profiling','University of Manchester',''],\n['[MirGeneDB](https:\u002F\u002Fmirgenedb.org)','Epigenomics','Multiple (Metazoa)','miRNA expression profile','RNAcentral group',''],\n['[dbSNP](https:\u002F\u002Fwww.ncbi.nlm.nih.gov\u002Fsnp\u002F)','Genomics','Multiple','single nucleotide variation','National Center for Biotechnology Information (NIH)',''],\n['[UniBind](https:\u002F\u002Funibind.uio.no)','Epigenomics','A. thaliana, C. elegans, D. rerio, D. melanogaster, H. sapiens, M. musculus, R. novergicus, S. cerevisiae, S. pombe ','transcription factor binding','Unibind group',''],\n['[ReMap](https:\u002F\u002Fremap.univ-amu.fr)','Epigenomics','H. sapiens, M. musculus, D. melanogaster, A. thaliana','transcription factor binding','ReMap group','Reprocessed data from GEO and ENCODE'],\n['[GTRD](http:\u002F\u002Fgtrd.biouml.org)','Epigenomics','A. thaliana, C. elegans, D. rerio, D. melanogaster, G. gallus, H. sapiens, M. musculus, R. novergicus, S. cerevisiae, S. pombe','transcription factor binding \u002F chromatin accessibility \u002F RNA initiation','BioUML',''],\n['[GeneBank](https:\u002F\u002Fwww.ncbi.nlm.nih.gov\u002Fgenbank\u002F)\u002F[DDBJ](https:\u002F\u002Fwww.ddbj.nig.ac.jp\u002Fddbj\u002F)\u002F[ENA](http:\u002F\u002Fwww.ebi.ac.uk\u002Fena\u002F)','Genomics','Multiple','DNA sequence','National Center for Biotechnology Information (NIH) \u002F DNA DataBank of Japan \u002F European Bioinformatics Institute',],\n['[Expression Atlas](https:\u002F\u002Fwww.ebi.ac.uk\u002Fgxa\u002F)','Genomics','Multiple','RNA profile expression','European Bioinformatics Institute','Reprocess data from COVID-19 Data portal, ENCODE, BLUEPRINT, FANTOM, the Human Atlas Project, Cancer Cell Line Encyclopedia, HipSci, GTEx, PCAWG, Genomics of Drug Sensitivity in Cancer Project, HDBR, Gramene'],\n['[FlyBase](http:\u002F\u002Fflybase.org)','Genomics \u002F Epigenomics','D. simulans, D. sechellia, D. melanogaster, D. yakuba, D. erecta, D. ananassae, D. pseudoobscura, D. persimillis, D. willistoni, D. mojavensis, D. virillis, D. grimshawi','DNA sequence \u002F DNA sequence variation \u002F mRNA profile expression \u002F ncRNA profile expression \u002F 3D chromatin structure \u002F protein sequence','FlyBase consortium ','Data in RNA central, part of [Alliance of Genome Resource](https:\u002F\u002Fwww.alliancegenome.org)'],\n['[Mouse Genome Informatics (MGI)](http:\u002F\u002Fwww.informatics.jax.org )','Genomics \u002F Epigenomics','M. musculus','DNA sequence \u002F RNA expression profile','MGI project','Data in RNA central, part of [Alliance of Genome Resource](https:\u002F\u002Fwww.alliancegenome.org)'],\n['[Rat Genome Database (RGD)](https:\u002F\u002Frgd.mcw.edu)','Genomics \u002F Epigenomics',' R. norvegicus, M. musculus, H. sapiens, C. lanigera, P. paniscusm, C. familiaris, I. tridecemlineatus, S. scrofa, H. glaber, C. sabaeus','DNA sequence \u002F RNA expression profile','RGD group (NIH)','Data in RNA central, part of [Alliance of Genome Resource](https:\u002F\u002Fwww.alliancegenome.org)'],\n['[WormBase](https:\u002F\u002Fwormbase.org)','Genomics \u002F Epigenomics','C. elegans and related nematodes','DNA sequence \u002F RNA expression profile','WormBase consortium','Data in RNA central, part of [Alliance of Genome Resource](https:\u002F\u002Fwww.alliancegenome.org)'],\n['[ZFIN](http:\u002F\u002Fzfin.org )','Genomics \u002F Epigenomics','D. rerio','DNA sequence \u002F RNA expression profile','Zebrafish Information Network group','Data in RNA central, part of [Alliance of Genome Resource](https:\u002F\u002Fwww.alliancegenome.org)'],\n['[Saccharomyces Genome Database (SGD)](https:\u002F\u002Fyeastgenome.org)','Genomics \u002F Epigenomics','S. cerevisiae','DNA sequence \u002F RNA expression profile','SGD group','Data in RNA central, part of [Alliance of Genome Resource](https:\u002F\u002Fwww.alliancegenome.org)'],\n['[Xenbase](http:\u002F\u002Fwww.xenbase.org\u002F)','Genomics \u002F Epigenomics','X. laevis, X. tropicalis','DNA sequence \u002F RNA expression profile \u002F protein sequence \u002F tanscription sequence binding','Xenbase group',],\n['[The Arabidopsis Information Resource (TAIR)](https:\u002F\u002Fwww.arabidopsis.org )','Genomics','A. thaliana','DNA sequence \u002F RNA expression profile','TAIR group','Data in RNA central'],\n['[Influenza Research Database (IRD)](https:\u002F\u002Fwww.fludb.org)','Genomics \u002F Disease','Influenza virus, hosts','DNA sequence \u002F 3D protein structure','IRD group',''],\n['[VEuPathDB](https:\u002F\u002Fvectorbase.org)','Genomics \u002F Disease','Eukaryotic pathogens and invertebrate vectors of disease','DNA sequence','Eukaryotic Pathogen, Vector and Host Informatics Resource',''],\n]",":table-number":O,":table-caption":Y},children:[{type:a,value:P}]}]},dir:s,path:"\u002Ftopics\u002Fs05-track-collections",extension:t,createdAt:q,updatedAt:q},{slug:"s06-analysing-tracks",title:"Analysing Tracks",img:x,caption:l,ingress:l,toc:[],body:{type:r,children:[{type:b,tag:d,props:{},children:[{type:a,value:l}]}]},dir:s,path:"\u002Ftopics\u002Fs06-analysing-tracks",extension:t,createdAt:q,updatedAt:q},{slug:"s07-fairification",title:"FAIRification of Track Metadata",img:x,caption:l,ingress:l,toc:[],body:{type:r,children:[{type:b,tag:d,props:{},children:[{type:a,value:l}]}]},dir:s,path:"\u002Ftopics\u002Fs07-fairification",extension:t,createdAt:q,updatedAt:v},{slug:"s08-identifiers",title:af,img:x,caption:l,ingress:l,toc:[],body:{type:r,children:[{type:b,tag:d,props:{},children:[{type:a,value:"A Globally Unique Identifier (GUID), also referred to as Universal Unique Identifier (UUID),\nis a 128-bit unique reference number that identifies unique Internet objects or data.\nA GUID is highly unlikely to repeat when generated despite the absence of a central GUID authority to ensure uniqueness."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"There are few solutions for assigning globally unique identifiers to track files. In many cases, only the raw sequence files are assigned identifiers,\ntypically accession number to core data repositories, such as the Sequence Read Archive (SRA)26 or the European Genome-Phenome Archive (EGA)23.\nThe ENCODE10 project represents a welcome exception, as they have assigned Identifiers.org-resolvable identifiers to each single track file with associated web-pages.\nTrack files typically contain condensed data returned from bioinformatics tools, which are run as part of defined workflows.\nAs such, the track files are prone to contain biases due to specific parameter settings or workflow states,\nwarranting the common advice to researchers to redo all analyses from the sequence files, with full control of the complete process.\nHowever, this might be impractical or unfeasible. Furthermore, track analysis is often exploratory in nature and a certain amount of error is often acceptable.\nThe ability to uniquely identify specific track files is thus crucial for track analysis with positive consequences for reproducibility27.\nA globally accessible service to assign and register identifiers to single track files and collections of tracks is currently missing.\nWe strongly recommend the implementation of a track registry that also preserves the full context surrounding the track files, with global identifiers to not only the track\nfiles, but also to the associated metadata. Our FAIRtracks draft standard could be advantageous for use as a basis for such a registry of track metadata.\nFor now, we are leveraging the widespread adoption of the document identifier (DOI) by requiring a FAIRtracks document to be published and identified with a DOI.\nWe require the publisher to support DOI versioning and also the possibility of reserving a DOI prior to publication (to include the DOI in the published file itself).\nWe currently recommend using Zenodoxxxix as the publishing platform, as the service supports both features,\nbut other platforms are also suitable as long as both DOI versioning and reservation are possible.\nFAIRtracks is easily extendable to support other global identifier types. To improve the provenance of analyses making use of genomic track data, e.g., as part of a publication,\nit would be useful to also be able to uniquely identify specific ad hoc collections of tracks.\nOne way to support this would be to only allow a single track collection in a FAIRtracks document, and then refer to the DOI.\nThere is, however, an issue with granularity, as the number of published datasets in that case would be impractically high,\nas no publishing platform supports identifiers at a sub-dataset level in combination with versioning and pre-reservation of DOIs.\nCreating a separate service to assign and register identifiers at the track collection level seems a\nbetter solution (see A2 below)."}]}]},dir:s,path:"\u002Ftopics\u002Fs08-identifiers",extension:t,createdAt:v,updatedAt:v},{slug:ag,title:ah,img:am,caption:"Phagocyte definition according to [Cell Ontology](http:\u002F\u002Fwww.ontobee.org\u002Fontology\u002FCL?iri=http:\u002F\u002Fpurl.obolibrary.org\u002Fobo\u002FCL_0000234).",ingress:"Biological terms used to describe metadata must be formalised",toc:[],body:{type:r,children:[{type:b,tag:d,props:{},children:[{type:a,value:"Ontologies are a formal way of representing knowledge in which concepts are described by their meaning and their\nrelationship. Ontologies assign to each concept (or term) its own unique identifier (ID), thus facilitating\narchiving, storing and accessing data in databases."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"For example, a phagocyte is a cell type represented by a term (or concept) with a description, an ID and\nrelationships. Its relationship defined it as a subclass of Native cell, Motile cell, stuff Accumulating cell, and\ndefensive cell; other terms with their own definition, ID and relationships (see diagram). A Cell Ontology allows\nthe definition of all of those concepts."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Interoperability across ontologies is invaluable to describe complex biological knowledge with composite\nannotations. Ontologies attribution of unique IDs to biological terms not only facilitates storing and searching\ndata in databases but also linking information between them. Using the previous example, the phagocyte cell type as\na unique ID in "},{type:b,tag:e,props:{href:"https:\u002F\u002Fobofoundry.org\u002Fontology\u002Fcl.html",rel:[g,h,i],target:j},children:[{type:a,value:"Cell Ontology"}]},{type:a,value:", but also foreign ID linking it to the\n"},{type:b,tag:e,props:{href:"http:\u002F\u002Fgeneontology.org",rel:[g,h,i],target:j},children:[{type:a,value:"Gene Ontology"}]},{type:a,value:" describing phagocytosis as a biological process linked to the cell."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"To support interoperability, "},{type:b,tag:e,props:{href:"https:\u002F\u002Fobofoundry.org",rel:[g,h,i],target:j},children:[{type:a,value:"OBO foundry"}]},{type:a,value:" is a community whose goal is to provide a set of\ninteroperable ontologies that are both logically well-formed and scientifically accurate. They review participating\nontologies according to sets of "},{type:b,tag:e,props:{href:"https:\u002F\u002Fobofoundry.org\u002Fprinciples\u002Ffp-000-summary.html",rel:[g,h,i],target:j},children:[{type:a,value:"principles and frameworks"}]},{type:a,value:".\n"},{type:b,tag:e,props:{href:"https:\u002F\u002Fbioportal.bioontology.org",rel:[g,h,i],target:j},children:[{type:a,value:"BioPortal"}]},{type:a,value:" proposes a service to access OBO foundry's complying ontologies."}]}]},dir:s,path:"\u002Ftopics\u002Fs09-ontologies",extension:t,createdAt:v,updatedAt:v},{slug:"s10-validation",title:"Validation",img:x,caption:l,ingress:l,toc:[],body:{type:r,children:[{type:b,tag:d,props:{},children:[{type:a,value:l}]}]},dir:s,path:"\u002Ftopics\u002Fs10-validation",extension:t,createdAt:v,updatedAt:v},{slug:"s11-file_formats",title:"File Formats",img:x,caption:l,ingress:l,toc:[],body:{type:r,children:[{type:b,tag:d,props:{},children:[{type:a,value:"A track format aims to allow systematic handling of genomic data and metadata by standardizing information contained in files. A critical purpose of a track format is to be easily usable and sharable by humans and machines. In order to do that, they codify what minimal information must be present in a file and its formatting.\nGenerally, file formats are specific to distinct types of genomic information: genomic sequence, genomic variation, genomic expression, genomic ranges and genomic structure. A good overview of the most used data file format is available for data sharing is available at "},{type:b,tag:e,props:{href:"https:\u002F\u002Fgenome.ucsc.edu\u002FFAQ\u002FFAQformat.html#format1",rel:[g,h,i],target:j},children:[{type:a,value:"UCSC genome browser documentation"}]}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Although some consensus exists, a complete overview of all file formats is problematized by the number available. A wide variety of genomic information can be represented, from genomic sequences to complex three-dimensional structures (see Track Types topic), making the development of a single file format standard difficult. Currently, several formats exist, covering all types of genomic annotations, with some formats describing the same information with different standards. Moreover, variations do exist. For example, BED file formats have several variations, generally named BED X + Y, where X is the number of standard columns, and Y is the number of columns containing non-standard information. Those variations mean that files following the same file standard might not be compatible. It also adds to the plethora of already existing formats leads to a proliferation of formats that complicate interoperability and reusability of data."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"File formats cover not only data and metadata present in a file but also computational file formats. Plain text format ensures human readability at the expense of file size. The quantity of information provided negatively impacts genomic track files size, complicating their programmatic manipulation. Several file formats have lossless compressed versions that are indexed binary variations such as BAM for SAM or BigBED for BED to address this issue. Several tools have been developed to convert and manipulate plain text and binary files interchangeably, such as "},{type:b,tag:e,props:{href:"https:\u002F\u002Fbedtools.readthedocs.io",rel:[g,h,i],target:j},children:[{type:a,value:"bedtools"}]},{type:a,value:E},{type:b,tag:e,props:{href:"http:\u002F\u002Fsamtools.sourceforge.net",rel:[g,h,i],target:j},children:[{type:a,value:"samtools"}]},{type:a,value:w}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Standardization of genomic files format should be done at every step of a project and, ideally, be decided before data generation\u002Fgathering. This measure is even more critical for collaborative studies and projects where data generation and analysis is shared between different groups. International consortiums like "},{type:b,tag:e,props:{href:"https:\u002F\u002Fwww.encodeproject.org\u002Fhelp\u002Ffile-formats\u002F",rel:[g,h,i],target:j},children:[{type:a,value:"ENCODE"}]},{type:a,value:" and BLUEPRINT have dedicated part of their preliminary works to codify files formats for data and metadata. At a higher level, there is an international effort to select and standardize current formats to ensure the perenization of data and metadata. The "},{type:b,tag:e,props:{href:"https:\u002F\u002Fwww.ga4gh.org",rel:[g,h,i],target:j},children:[{type:a,value:"Global Alliance for Genomics and Health"}]},{type:a,value:" (GA4GH) is an alliance whose purpose is to \"cultivate a common framework of standards and harmonized approaches for effective and responsible genomic and health-related data sharing\". One of their work is to access and adopt open standards for genomic data sharing and provide guidance and tools to complete this goal ("},{type:b,tag:e,props:{href:an,rel:[g,h,i],target:j},children:[{type:a,value:an}]},{type:a,value:")."}]}]},dir:s,path:"\u002Ftopics\u002Fs11-file_formats",extension:t,createdAt:v,updatedAt:v},{slug:"s12-interoperability",title:"FAIR interoperability",img:x,caption:l,ingress:l,toc:[],body:{type:r,children:[{type:b,tag:d,props:{},children:[{type:a,value:l}]}]},dir:s,path:"\u002Ftopics\u002Fs12-interoperability",extension:t,createdAt:v,updatedAt:v}],imageAssetObjects:{"genomic-tracks.jpeg":{filename:T,isSvgImage:z,responsiveImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd56a02a-160.jpeg 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Faa55c3c-320.jpeg 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F101c100-640.jpeg 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1614d19-800.jpeg 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F4027e5b-960.jpeg 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F32f32a7-1280.jpeg 1280w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F385c3c6-1440.jpeg 1440w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F7a5e490-1600.jpeg 1600w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd3b3879-1920.jpeg 1920w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F78de58a-2560.jpeg 2560w",images:[{path:ao,width:k,height:A},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Faa55c3c-320.jpeg",width:u,height:F},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F101c100-640.jpeg",width:n,height:ap},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1614d19-800.jpeg",width:o,height:aq},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F4027e5b-960.jpeg",width:y,height:B},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F32f32a7-1280.jpeg",width:B,height:ar},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F385c3c6-1440.jpeg",width:G,height:C},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F7a5e490-1600.jpeg",width:H,height:as},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd3b3879-1920.jpeg",width:C,height:I},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F78de58a-2560.jpeg",width:I,height:at}],src:ao,width:k,height:A},responsiveWebpImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F78c3436-160.webp 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fe7b1492-320.webp 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd71c310-640.webp 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fcb8bb57-800.webp 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F835fea1-960.webp 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F65542f0-1280.webp 1280w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ffd96b23-1440.webp 1440w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F93a53c5-1600.webp 1600w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F17342e4-1920.webp 1920w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F4820a1f-2560.webp 2560w",images:[{path:au,width:k,height:A},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fe7b1492-320.webp",width:u,height:F},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd71c310-640.webp",width:n,height:ap},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fcb8bb57-800.webp",width:o,height:aq},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F835fea1-960.webp",width:y,height:B},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F65542f0-1280.webp",width:B,height:ar},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ffd96b23-1440.webp",width:G,height:C},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F93a53c5-1600.webp",width:H,height:as},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F17342e4-1920.webp",width:C,height:I},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F4820a1f-2560.webp",width:I,height:at}],src:au,width:k,height:A},optimizedImagePath:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F40fa760.jpeg",placeholderImagePath:Object("data:image\u002Fsvg+xml,%3Csvg xmlns='http:\u002F\u002Fwww.w3.org\u002F2000\u002Fsvg' viewBox='0 0 2655 3540'%3E%3Cfilter id='b'%3E%3CfeGaussianBlur stdDeviation='12' \u002F%3E%3C\u002Ffilter%3E%3Cpath fill='%23f3f3ee' d='M0 0h2655v3540H0z'\u002F%3E%3Cg filter='url(%23b)' transform='translate(7 7) scale(13.82813)' fill-opacity='.5'%3E%3Cpath fill='%23202517' d='M9 196h125v12H9z'\u002F%3E%3Cpath fill='%23acb32e' d='M148 94l35-18-37-59z'\u002F%3E%3Cellipse fill='%23505800' cx='107' cy='114' rx='36' ry='4'\u002F%3E%3Cellipse fill='%23798220' rx='1' ry='1' transform='rotate(83.2 -16.6 145.8) scale(3.81623 37.44377)'\u002F%3E%3Cpath fill='%232a1912' d='M9.3 194.9v10.9l23.1-14.5L66.8 171z'\u002F%3E%3Cellipse fill='%23798e83' rx='1' ry='1' transform='matrix(.94624 -3.93106 52.67165 12.67846 116.7 210.2)'\u002F%3E%3Cpath fill='%23818d23' d='M119 110l41 30-5-21z'\u002F%3E%3Cellipse fill='%23fff' rx='1' ry='1' transform='rotate(-164.2 14.1 13.4) scale(103.80477 162.36772)'\u002F%3E%3Cpath fill='%23495011' d='M61.5 97.7L75 76.8l-7 6 7.6 34.3z'\u002F%3E%3Cpath fill='%23868f00' d='M151 51l-19-13 1-15z'\u002F%3E%3Cellipse fill='%23fff' cy='233' rx='162' ry='29'\u002F%3E%3Cpath fill='%232d2318' d='M23 206.6l-15.5-5.4 4.4-4 82.3 5.5z'\u002F%3E%3Cellipse fill='%23fff' cx='71' cy='196' rx='46' ry='2'\u002F%3E%3Cpath fill='%2387a39c' d='M131 208.2l28.6 17.8L79 202.2l94-8z'\u002F%3E%3Cpath fill='%23fff' d='M175 82.4l-19.6-30.6.8 15.5 50.5 97.1z'\u002F%3E%3Cellipse fill='%23fff' cx='184' cy='178' rx='28' ry='99'\u002F%3E%3Cpath fill='%23959b6e' d='M89.2 155.2l-12.2 23-6.2-3.4 12.2-23z'\u002F%3E%3Cellipse fill='%23485100' rx='1' ry='1' transform='matrix(8.26787 -8.44287 .71447 .69966 78.4 70.8)'\u002F%3E%3Cpath fill='%23764c3c' d='M69 208h61v3H69z'\u002F%3E%3Cpath fill='%23000500' d='M160 217.4l-10.8 8.8-1.3-1.6 11-8.8z'\u002F%3E%3C\u002Fg%3E%3C\u002Fsvg%3E")},"genome-browsers.png":{filename:W,isSvgImage:z,responsiveImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1fc8819-160.png 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1143b40-320.png 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fe8ccd21-640.png 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fdb55537-800.png 800w",images:[{path:av,width:k,height:J},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1143b40-320.png",width:u,height:aw},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fe8ccd21-640.png",width:n,height:ax},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fdb55537-800.png",width:o,height:ay}],src:av,width:k,height:J},responsiveWebpImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fc1517b1-160.webp 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fea2fbb6-320.webp 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fbf74273-640.webp 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff9f087b-800.webp 800w",images:[{path:az,width:k,height:J},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fea2fbb6-320.webp",width:u,height:aw},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fbf74273-640.webp",width:n,height:ax},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff9f087b-800.webp",width:o,height:ay}],src:az,width:k,height:J},optimizedImagePath:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff52c6a9.png",placeholderImagePath:Object("data:image\u002Fsvg+xml,%3Csvg xmlns='http:\u002F\u002Fwww.w3.org\u002F2000\u002Fsvg' viewBox='0 0 800 450'%3E%3Cfilter id='b'%3E%3CfeGaussianBlur stdDeviation='12' \u002F%3E%3C\u002Ffilter%3E%3Cpath fill='%23dbdbe4' d='M0 0h800v450H0z'\u002F%3E%3Cg filter='url(%23b)' transform='translate(1.6 1.6) scale(3.125)' fill-opacity='.5'%3E%3Cellipse cx='171' cy='118' rx='140' ry='6'\u002F%3E%3Cellipse fill='%23000063' cx='157' cy='117' rx='116' ry='6'\u002F%3E%3Cpath fill='%23fff' d='M74.2-18.5l-34.4 195-68-12 34.4-195z'\u002F%3E%3Cellipse fill='%23f7f7f9' cx='224' cy='46' rx='77' ry='68'\u002F%3E%3Cpath fill='%23000060' d='M51 113h87v11H51z'\u002F%3E%3Cpath fill='%23fff' d='M30 115l-4-74 78-4 3.8 74z'\u002F%3E%3Cpath fill='%237b7b77' d='M118 136h71v8h-71z'\u002F%3E%3Cpath fill='%2321211d' d='M73.3 29.8l.4 10-13 .4-.4-10z'\u002F%3E%3Cpath d='M195 30h7v5h-7z'\u002F%3E%3Cellipse fill='%2300004a' cx='55' cy='18' rx='22' ry='2'\u002F%3E%3Cellipse fill='%23fffffd' cx='21' cy='122' rx='29' ry='177'\u002F%3E%3Cpath fill='%23373731' d='M239.6 146.4l-35 1.2-.2-8 35-1.2z'\u002F%3E%3Cpath fill='%23000068' d='M131 113h125v11H131z'\u002F%3E%3Cpath d='M75 139h19v5H75z'\u002F%3E%3Cpath fill='%236c88ad' d='M151.1 79.5l-33-15.7-6 26 22.7-20.2z'\u002F%3E%3Cpath fill='%23fffffa' d='M0 35h166v18H0z'\u002F%3E%3Cellipse fill='%234c4883' cx='181' cy='56' rx='118' ry='1'\u002F%3E%3Cpath d='M135 139h7v5h-7z'\u002F%3E%3Cpath fill='%23787879' d='M123.4 37.6l8.7-12.8 2.4 15-62.2-.6z'\u002F%3E%3Cpath fill='%23fff' d='M49 124h57v15H49z'\u002F%3E%3C\u002Fg%3E%3C\u002Fsvg%3E")},"track-types.png":{filename:Z,isSvgImage:z,responsiveImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fea8db5a-160.png 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2f69796-320.png 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F46bea7c-640.png 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1525edf-800.png 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff48bcaa-911.png 911w",images:[{path:aA,width:k,height:K},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2f69796-320.png",width:u,height:aB},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F46bea7c-640.png",width:n,height:aC},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1525edf-800.png",width:o,height:aD},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff48bcaa-911.png",width:aE,height:aF}],src:aA,width:k,height:K},responsiveWebpImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F9a374d1-160.webp 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F68f5807-320.webp 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2664ade-640.webp 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F035cee0-800.webp 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3dceba8-911.webp 911w",images:[{path:aG,width:k,height:K},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F68f5807-320.webp",width:u,height:aB},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2664ade-640.webp",width:n,height:aC},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F035cee0-800.webp",width:o,height:aD},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3dceba8-911.webp",width:aE,height:aF}],src:aG,width:k,height:K},optimizedImagePath:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F65ca73a.png",placeholderImagePath:Object("data:image\u002Fsvg+xml,%3Csvg xmlns='http:\u002F\u002Fwww.w3.org\u002F2000\u002Fsvg' viewBox='0 0 911 1285'%3E%3Cfilter id='b'%3E%3CfeGaussianBlur stdDeviation='12' \u002F%3E%3C\u002Ffilter%3E%3Cpath fill='%23f5f4ed' d='M0 0h908v1285H0z'\u002F%3E%3Cg filter='url(%23b)' transform='translate(2.5 2.5) scale(5.01953)' fill-opacity='.5'%3E%3Cellipse fill='%23b9ab20' cx='149' cy='146' rx='49' ry='4'\u002F%3E%3Cellipse fill='%23cdc14b' cx='145' cy='175' rx='40' ry='6'\u002F%3E%3Cellipse fill='%23c9bd3f' cx='55' cy='147' rx='165' ry='3'\u002F%3E%3Cellipse fill='%23c5b834' cx='21' cy='176' rx='57' ry='4'\u002F%3E%3Cpath fill='%23d3c63e' d='M-13.4 208.5l77.8 6.5-27-14-16.2.8z'\u002F%3E%3Cellipse fill='%23fff' cx='90' cy='85' rx='21' ry='174'\u002F%3E%3Cpath fill='%23cbbb1b' d='M170 211l-33-12-36 9z'\u002F%3E%3Cellipse fill='%23a3a4aa' cx='141' cy='239' rx='36' ry='6'\u002F%3E%3Cpath fill='%23cbbc2b' d='M103 115l35-12-3 11z'\u002F%3E%3Cpath fill='%23c4b100' d='M16 108h17v9H16z'\u002F%3E%3Cellipse fill='%23fefeff' cx='62' cy='35' rx='178' ry='47'\u002F%3E%3Cpath fill='%23b2a523' d='M16 82h20v6H16z'\u002F%3E%3Cellipse fill='%23fff' rx='1' ry='1' transform='matrix(62.47964 11.01685 -4.40766 24.99706 40.5 237.2)'\u002F%3E%3Cellipse fill='%23b4a93f' cx='124' cy='85' rx='14' ry='4'\u002F%3E%3Cpath fill='%235a5b5e' d='M118 3h39v2h-39z'\u002F%3E%3Cpath fill='%23e2d872' d='M61.7 186.4l-47.7-11 2.3-9.8 47.7 11z'\u002F%3E%3Cpath fill='%23b6ac3d' d='M185 208.3l-31.2 7 2.5-3.8 16.3-6.2z'\u002F%3E%3Cpath fill='%23343437' d='M26 3h25v2H26z'\u002F%3E%3Cpath fill='%23bfb33f' d='M2 144h73v6H2z'\u002F%3E%3Cellipse fill='%23fff' cx='164' cy='139' rx='180' ry='5'\u002F%3E%3C\u002Fg%3E%3C\u002Fsvg%3E")},"under-construction.png":{filename:x,isSvgImage:z,responsiveImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2dc532f-160.png 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fe72cc86-320.png 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fbce2fe5-640.png 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff680422-800.png 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fc607bde-960.png 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F5e5e4ca-1280.png 1280w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fa2c6e33-1440.png 1440w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fb9f391e-1600.png 1600w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F9bab1b4-1920.png 1920w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2f174fc-2019.png 2019w",images:[{path:aH,width:k,height:L},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fe72cc86-320.png",width:u,height:aI},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fbce2fe5-640.png",width:n,height:aJ},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff680422-800.png",width:o,height:aK},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fc607bde-960.png",width:y,height:aL},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F5e5e4ca-1280.png",width:B,height:aM},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fa2c6e33-1440.png",width:G,height:aN},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fb9f391e-1600.png",width:H,height:aO},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F9bab1b4-1920.png",width:C,height:aP},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2f174fc-2019.png",width:aQ,height:aR}],src:aH,width:k,height:L},responsiveWebpImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1afec95-160.webp 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F0bce009-320.webp 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F4ed0e34-640.webp 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd70f224-800.webp 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F90091a1-960.webp 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3c6566f-1280.webp 1280w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F96af9fe-1440.webp 1440w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F64e1327-1600.webp 1600w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F5f22ed2-1920.webp 1920w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd58cb0e-2019.webp 2019w",images:[{path:aS,width:k,height:L},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F0bce009-320.webp",width:u,height:aI},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F4ed0e34-640.webp",width:n,height:aJ},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd70f224-800.webp",width:o,height:aK},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F90091a1-960.webp",width:y,height:aL},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3c6566f-1280.webp",width:B,height:aM},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F96af9fe-1440.webp",width:G,height:aN},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F64e1327-1600.webp",width:H,height:aO},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F5f22ed2-1920.webp",width:C,height:aP},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd58cb0e-2019.webp",width:aQ,height:aR}],src:aS,width:k,height:L},optimizedImagePath:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd07df6d.png",placeholderImagePath:Object("data:image\u002Fsvg+xml,%3Csvg xmlns='http:\u002F\u002Fwww.w3.org\u002F2000\u002Fsvg' viewBox='0 0 2019 2400'%3E%3Cfilter id='b'%3E%3CfeGaussianBlur stdDeviation='12' \u002F%3E%3C\u002Ffilter%3E%3Cpath fill='%238f3d00' d='M0 0h2015v2400H0z'\u002F%3E%3Cg filter='url(%23b)' transform='translate(4.7 4.7) scale(9.375)' fill-opacity='.5'%3E%3Cellipse rx='1' ry='1' transform='matrix(-34.5715 -18.9502 94.4701 -172.34511 29.4 47)'\u002F%3E%3Cellipse rx='1' ry='1' transform='rotate(152.9 91.2 39.1) scale(49.47142 213.99999)'\u002F%3E%3Cellipse fill='%23ff8f00' cx='108' cy='201' rx='106' ry='106'\u002F%3E%3Cellipse fill='%23ff6800' rx='1' ry='1' transform='matrix(25.87583 -56.77932 32.0738 14.61688 97.1 76.2)'\u002F%3E%3Cellipse fill='%23010' rx='1' ry='1' transform='matrix(-11.91695 30.2685 -14.81379 -5.8323 115 125.1)'\u002F%3E%3Cellipse fill='%23de0000' rx='1' ry='1' transform='matrix(183.24252 -.1281 .00962 13.76654 104.5 180.1)'\u002F%3E%3Cellipse rx='1' ry='1' transform='matrix(80.84157 -186.81385 38.90037 16.8337 34 20.3)'\u002F%3E%3Cellipse rx='1' ry='1' transform='matrix(-99.52578 -167.30734 34.39047 -20.45779 191 53)'\u002F%3E%3Cpath fill='%23ffe700' d='M4 195h64v57H4z'\u002F%3E%3Cellipse fill='red' rx='1' ry='1' transform='matrix(66.71392 113.75798 -11.01953 6.46245 149 84.8)'\u002F%3E%3Cellipse fill='%23ff0' rx='1' ry='1' transform='matrix(31.88075 -1.0305 .51187 15.83589 178.6 211.3)'\u002F%3E%3Cellipse fill='red' rx='1' ry='1' transform='matrix(-11.3422 -6.48434 85.81362 -150.10229 46.6 118.5)'\u002F%3E%3Cpath fill='%23ff0' d='M107 32l52 85-72-49z'\u002F%3E%3Cellipse fill='%23ff0' cx='148' cy='144' rx='22' ry='23'\u002F%3E%3Cellipse cx='26' cy='191' rx='214' ry='4'\u002F%3E%3Cellipse fill='%23444f00' cx='96' cy='235' rx='102' ry='9'\u002F%3E%3Cpath fill='%23f8d700' d='M24 173l72-21 10-111z'\u002F%3E%3Cellipse cx='132' cy='255' rx='185' ry='3'\u002F%3E%3Cpath fill='%23413f00' d='M80 112h36v47H80z'\u002F%3E%3Cellipse fill='red' cx='96' cy='178' rx='212' ry='11'\u002F%3E%3C\u002Fg%3E%3C\u002Fsvg%3E")},"track-collections.png":{filename:ai,isSvgImage:z,responsiveImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F81ad884-160.png 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fc1d756d-320.png 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1943e11-640.png 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F84acdbe-800.png 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fbc3d18b-960.png 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3d88da6-1200.png 1200w",images:[{path:aT,width:k,height:M},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fc1d756d-320.png",width:u,height:A},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1943e11-640.png",width:n,height:F},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F84acdbe-800.png",width:o,height:aU},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fbc3d18b-960.png",width:y,height:n},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3d88da6-1200.png",width:aV,height:o}],src:aT,width:k,height:M},responsiveWebpImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F0bc2586-160.webp 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F74a7104-320.webp 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F22230c1-640.webp 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fa0d266b-800.webp 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fcc77733-960.webp 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F4233362-1200.webp 1200w",images:[{path:aW,width:k,height:M},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F74a7104-320.webp",width:u,height:A},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F22230c1-640.webp",width:n,height:F},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fa0d266b-800.webp",width:o,height:aU},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fcc77733-960.webp",width:y,height:n},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F4233362-1200.webp",width:aV,height:o}],src:aW,width:k,height:M},optimizedImagePath:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F359a83b.png",placeholderImagePath:Object("data:image\u002Fsvg+xml,%3Csvg xmlns='http:\u002F\u002Fwww.w3.org\u002F2000\u002Fsvg' viewBox='0 0 1200 800'%3E%3Cfilter id='b'%3E%3CfeGaussianBlur stdDeviation='12' \u002F%3E%3C\u002Ffilter%3E%3Cpath fill='%23f0ede9' d='M0 0h1200v796H0z'\u002F%3E%3Cg filter='url(%23b)' transform='translate(2.3 2.3) scale(4.6875)' fill-opacity='.5'%3E%3Cellipse fill='%23c1987f' rx='1' ry='1' transform='matrix(-74.64866 -13.83881 4.65874 -25.12997 100.3 106.4)'\u002F%3E%3Cpath fill='%23ffa026' d='M51 47h19v87H51z'\u002F%3E%3Cpath fill='%2385b9fb' d='M28.4 44.8L47 31.4l1 102-20.2 4.2z'\u002F%3E%3Cellipse fill='%23fff' rx='1' ry='1' transform='matrix(172.60783 64.87902 -23.4977 62.51462 158 22.7)'\u002F%3E%3Cpath fill='%2360f866' d='M73 76h18v59H73z'\u002F%3E%3Cpath fill='%239fc6a2' d='M187 112h64v22h-64z'\u002F%3E%3Cellipse fill='%23acaeb0' cx='169' cy='8' rx='125' ry='3'\u002F%3E%3Cellipse fill='%23fff' cx='240' cy='45' rx='171' ry='40'\u002F%3E%3Cpath fill='%23ff6f7a' d='M96 83h19v51H96z'\u002F%3E%3Cellipse fill='%23fff' rx='1' ry='1' transform='matrix(18.57395 88.1397 -22.58013 4.75838 0 61.9)'\u002F%3E%3Cpath fill='%23d9d1ff' d='M115 83h26v58h-26z'\u002F%3E%3Cellipse fill='%23a1a0a0' cx='184' cy='153' rx='189' ry='2'\u002F%3E%3Cellipse fill='%23b27a79' cx='150' cy='111' rx='10' ry='20'\u002F%3E%3Cpath fill='%23ffa9e0' d='M160 103h23v31h-23z'\u002F%3E%3Cellipse fill='%23fff' cx='137' cy='141' rx='244' ry='8'\u002F%3E%3Cellipse fill='%23f4e36d' rx='1' ry='1' transform='matrix(10.61966 -1.83433 2.05506 11.89754 218.3 121.9)'\u002F%3E%3Cellipse fill='%23fff' cx='94' cy='96' rx='2' ry='54'\u002F%3E%3Cpath fill='%239ae8ff' d='M228 110h23v24h-23z'\u002F%3E%3Cpath fill='%23fff' d='M47 77h5v41h-5z'\u002F%3E%3Cellipse fill='%23fff' cx='151' cy='66' rx='50' ry='28'\u002F%3E%3C\u002Fg%3E%3C\u002Fsvg%3E")},"ontologies.png":{filename:am,isSvgImage:z,responsiveImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1ff4490-160.png 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fb1b7839-320.png 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3a2d08a-640.png 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F96019d5-800.png 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fb07c0e9-960.png 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F45a6864-961.png 961w",images:[{path:aX,width:k,height:N},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fb1b7839-320.png",width:u,height:aY},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3a2d08a-640.png",width:n,height:aZ},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F96019d5-800.png",width:o,height:a_},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fb07c0e9-960.png",width:y,height:a$},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F45a6864-961.png",width:ba,height:bb}],src:aX,width:k,height:N},responsiveWebpImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F87e8435-160.webp 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fade0757-320.webp 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fefb1e63-640.webp 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F8efc590-800.webp 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fad32acf-960.webp 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F6576c18-961.webp 961w",images:[{path:bc,width:k,height:N},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fade0757-320.webp",width:u,height:aY},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fefb1e63-640.webp",width:n,height:aZ},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F8efc590-800.webp",width:o,height:a_},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fad32acf-960.webp",width:y,height:a$},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F6576c18-961.webp",width:ba,height:bb}],src:bc,width:k,height:N},optimizedImagePath:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff2d7581.png",placeholderImagePath:Object("data:image\u002Fsvg+xml,%3Csvg xmlns='http:\u002F\u002Fwww.w3.org\u002F2000\u002Fsvg' viewBox='0 0 961 486'%3E%3Cfilter id='b'%3E%3CfeGaussianBlur stdDeviation='12' \u002F%3E%3C\u002Ffilter%3E%3Cpath fill='%23eff4f8' d='M0 0h961v484H0z'\u002F%3E%3Cg filter='url(%23b)' transform='translate(1.9 1.9) scale(3.7539)' fill-opacity='.5'%3E%3Cellipse fill='%23186094' cx='73' cy='115' rx='173' ry='5'\u002F%3E%3Cellipse fill='%23000f62' cx='220' cy='55' rx='25' ry='5'\u002F%3E%3Cpath fill='%23003a80' d='M76 51.2l19.5-4.1L98 58.8l-19.5 4.1z'\u002F%3E%3Cpath fill='%2300005a' d='M229.6 18.8l-21 .4-.2-8 21-.4z'\u002F%3E%3Cpath fill='%23fff' d='M125 0h74v129h-74z'\u002F%3E%3Cpath fill='%23004789' d='M166.6 120l-21.6-3 6.5-8.5 16.8 3.8z'\u002F%3E%3Cpath fill='%23002d77' d='M233.7 118.4l-23 1.2-.4-8 23-1.2z'\u002F%3E%3Cpath fill='%23002577' d='M82 11h10v9H82z'\u002F%3E%3Cpath fill='%23fff' d='M35.3 134.1L6.8-16l-22.2 160 98-123z'\u002F%3E%3Cellipse fill='%231f6191' rx='1' ry='1' transform='matrix(-.00523 -3.16238 57.78582 -.09551 64.7 115.5)'\u002F%3E%3Cpath fill='%23fff' d='M43.3 106L92-16-16 9.4 45 144zM87 74h10v55H87zm78.5 71.3l3-84 38 1.4-3 84z'\u002F%3E%3Cpath fill='%23006499' d='M196 51h45v8h-45z'\u002F%3E%3Cellipse fill='%23fff' cx='250' cy='115' rx='18' ry='70'\u002F%3E%3Cellipse fill='%2316547e' rx='1' ry='1' transform='rotate(-94.6 63.3 47.9) scale(5.38198 11.25844)'\u002F%3E%3Cpath fill='%23fff' d='M257 50L84 49l15-65z'\u002F%3E%3Cpath fill='%23fff' d='M125 43h20v86h-20z'\u002F%3E%3Cellipse fill='%23fff' cx='86' cy='128' rx='213' ry='10'\u002F%3E%3Cpath fill='%23fff' d='M60 59h196v52H60z'\u002F%3E%3C\u002Fg%3E%3C\u002Fsvg%3E")}}}],fetch:{},mutations:[]}}("text","element","\n","p","a","strong","nofollow","noopener","noreferrer","_blank",160,"TBA","em",640,800,"li","2022-03-31T10:38:45.957Z","root","\u002Ftopics",".md",320,"2022-03-31T10:38:45.961Z",".","under-construction.png",960,false,213,1280,1920,"lengths"," and ",427,1440,1600,2560,90,226,190,107,81,"1","\n\n\n","gaps","interconnections","nuxt-link","genomic-tracks.jpeg","Genomic tracks","ui-quote-text","genome-browsers.png","ui-markdown-table","'Various genome browsers deployed as web services with possibly domain-restricted track ' +\n'databases and\u002For available as installable software.'","track-types.png","et al."," ","https:\u002F\u002Fdoi.org\u002F10.1186\u002F1471-2105-12-494","(2011)","track types",", ","values","Identifiers","s09-ontologies","Ontologies","track-collections.png","https:\u002F\u002Fwww","https:\u002F\u002Frepositoryfinder.datacite.org","http:\u002F\u002Fgenome.ucsc.edu\u002Fcgi-bin\u002FhgHubConnect","ontologies.png","https:\u002F\u002Fwww.ga4gh.org\u002Fgenomic-data-toolkit\u002F","\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd56a02a-160.jpeg",853,1067,1707,2133,3413,"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F78c3436-160.webp","\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1fc8819-160.png",180,360,450,"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fc1517b1-160.webp","\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fea8db5a-160.png",451,903,1128,911,1285,"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F9a374d1-160.webp","\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2dc532f-160.png",380,761,951,1141,1522,1712,1902,2282,2019,2400,"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1afec95-160.webp","\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F81ad884-160.png",533,1200,"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F0bc2586-160.webp","\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1ff4490-160.png",162,324,405,485,961,486,"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F87e8435-160.webp")));