__NUXT_JSONP__("/topics", (function(a,b,c,d,e,f,g,h,i,j,k,l,m,n,o,p,q,r,s,t,u,v,w,x,y,z,A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,Q,R,S,T,U,V,W,X,Y,Z,_,$,aa,ab,ac,ad,ae,af,ag,ah,ai,aj,ak,al,am,an,ao,ap,aq,ar,as,at,au,av,aw,ax,ay,az,aA,aB,aC,aD,aE,aF,aG,aH,aI,aJ,aK,aL,aM,aN,aO,aP,aQ,aR,aS,aT,aU,aV,aW,aX,aY,aZ,a_,a$,ba,bb,bc,bd,be,bf,bg,bh,bi,bj,bk,bl,bm,bn){return {data:[{markdownFiles:[{slug:"s01-genomic-tracks",title:"Genomic Tracks",img:U,caption:"Unwinding of a DNA molecule from its compact chromatin form. Each nucleotide pair represents a coordinate position which can be annotated with genomic track data. (Image&nbsp;is&nbsp;[©&nbsp;scienceDISPLAY&nbsp;–&nbsp;stock.adobe.com](https:\u002F\u002Fstock.adobe.com\u002Fimages\u002Fdna-nucleosome-chromosom\u002F42940588))",ingress:"Tracks annotate the DNA with condensed data",toc:[],body:{type:u,children:[{type:b,tag:d,props:{},children:[{type:b,tag:q,props:{},children:[{type:a,value:V}]},{type:a,value:" refer to data files that annotate DNA reference sequence positions and can be visualized in genome\nbrowsers. Track files represent summaries of the raw data according to specific criteria and granularity.\nFor example: “hot spot” regions (with a high number of reads), values deviating from expectations, or cross-genomic\nlinks representing closeness in 3D. In essence, the condensed data in track files relate to the raw data much like an\nabstract describes a scientific publication. This data reduction allows researchers to scan large amounts of data to\ndefine a hypothesis before carrying out more accurate analyses.\n"},{type:b,tag:W,props:{":quote":"\"A genomic track relates to the raw data much like an abstract describes a scientific publication.\""},children:[{type:a,value:c}]},{type:a,value:"\nMany types of data from genomic analyses can be represented as genomic tracks, for example:"}]},{type:a,value:c},{type:b,tag:X,props:{},children:[{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Gene regions, repeating elements, conserved regions"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Chromatin accessibility (e.g., DNase I Hypersensitivity)"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Binding of Transcription Factors (TFs) to DNA"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Histone modifications along the DNA"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Gene expression, Gene fusions, Transcription Start Sites (TSS)"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Cis-regulatory elements (promoters, enhancers, ...)"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"DNA methylation"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"3D chromatin interaction"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Single Nucleotide Polymorphisms (SNPs) from Genome-Wide Association Studies (GWAS)"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Single Nucleotide Variants (SNVs) and Copy Number Variants (CNVs) in cancer cells"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"Virus insertion sites"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"plus many more"}]},{type:a,value:c}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Gradually, genomic tracks have also become a "},{type:b,tag:q,props:{},children:[{type:a,value:"de facto"}]},{type:a,value:" standard to store, distribute and analyze genome-wide datasets,\nmainly because of their efficient compression and indexing utilities. Many bioinformatics analyses are now being\ndistributed, either privately or publicly, using related file formats. Tracks are still predominantly used for graphical\ndisplay, but can also be queried by statistical analysis tools, such as\nthe "},{type:b,tag:e,props:{href:"https:\u002F\u002Fhyperbrowser.uio.no\u002F",rel:[g,h,i],target:j},children:[{type:a,value:"GSuite HyperBrowser"}]},{type:a,value:" , "},{type:b,tag:e,props:{href:"https:\u002F\u002Fblueprint-data.bsc.es\u002F",rel:[g,h,i],target:j},children:[{type:a,value:"EPICO"}]},{type:a,value:",\nor "},{type:b,tag:e,props:{href:"https:\u002F\u002Fdeepblue.mpi-inf.mpg.de\u002F",rel:[g,h,i],target:j},children:[{type:a,value:"DeepBlue"}]},{type:a,value:A}]}]},dir:v,path:"\u002Ftopics\u002Fs01-genomic-tracks",extension:w,createdAt:l,updatedAt:l},{slug:"s02-genome-browsers",title:"Genome Browsers",img:Y,caption:"Some different types of genomic tracks visualized in the [UCSC Genome Browser](https:\u002F\u002Fgenome.ucsc.edu\u002F)",ingress:"Genomic tracks are made for genome browsers",toc:[],body:{type:u,children:[{type:b,tag:d,props:{},children:[{type:b,tag:q,props:{},children:[{type:a,value:"Genomic track files"}]},{type:a,value:" were originally designed and optimized to be displayed within "},{type:b,tag:q,props:{},children:[{type:a,value:"genome browsers"}]},{type:a,value:". A genome browser\nis a type of genome-wide visualization software able to display various datasets as parallel tracks along the DNA\nsequence coordinates of typically reference genomes. A multitude of genome browsers are available as online web services\non top of a database of track files and\u002For as installable software. A selection genome browsers are listed below:"}]},{type:a,value:c},{type:b,tag:Z,props:{":headers":"['Genome Browser', 'Web service', 'Scientific domain', 'Installable software', 'Prog. language', 'Library']",":items":"[\n['[Ensembl](https:\u002F\u002Fwww.ensembl.org\u002F)', 'Yes', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002FEnsembl)', 'Perl \u002F Javascript', 'No'],\n['[UCSC Genome Browser](https:\u002F\u002Fgenome.ucsc.edu\u002F)',   'Yes', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002FucscGenomeBrowser\u002Fkent)', 'C \u002F HTML', 'No'],\n['[UCSC Xena](http:\u002F\u002Fxena.ucsc.edu\u002F)', 'Yes', 'Cancer multi-omics', '[Yes](https:\u002F\u002Fgithub.com\u002FucscXena)', 'Clojure \u002F Javascript', 'No'],\n['[WashU Epigenome Browser](https:\u002F\u002Fepigenomegateway.wustl.edu\u002F)', 'Yes', 'Epigenomics', '[Yes](https:\u002F\u002Fgithub.com\u002Ftwlab\u002Feg-react)', 'Javascript', 'No'],\n['[Zenbu \u002F FANTOM](https:\u002F\u002Ffantom.gsc.riken.jp\u002Fzenbu\u002F)','Yes', 'Gene regulation', '[Yes](https:\u002F\u002Fsourceforge.net\u002Fprojects\u002Fzenbu\u002F)', 'C++ \u002F Perl \u002F Javascript', 'No'],\n['[GTEx Locus Browser](https:\u002F\u002Fgtexportal.org\u002Fhome\u002FlocusBrowserPage\u002FACTN3)', 'Yes', 'Human variation (gene-centric)', '[Only demo](https:\u002F\u002Fgithub.com\u002Fbroadinstitute\u002Fgtex-viz\u002Ftree\u002Fmaster\u002FlocusBrowser)', 'Javascript', 'No'],\n['[DECIPHER](https:\u002F\u002Fwww.deciphergenomics.org\u002F)', 'Yes', 'Clinical data', 'No', '', ''],\n['[GTEx IGV Browser](https:\u002F\u002Fgtexportal.org\u002Fhome\u002FbrowseEqtls)', 'Yes', 'Human variation', 'No', '', ''],\n['[Plant Epigenome Browser](https:\u002F\u002Fepigenome.genetics.uga.edu\u002FPlantEpigenome\u002F)', 'Yes', 'Plant epigenomics', 'No', '', ''],\n['[Genoverse](https:\u002F\u002Fgenoverse.org\u002F)', 'No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002Fwtsi-web\u002FGenoverse)', 'Javascript', 'Yes'],\n['[GIVE](https:\u002F\u002Fzhong-lab-ucsd.github.io\u002FGIVE_homepage\u002F)', 'No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002FZhong-Lab-UCSD\u002FGenomic-Interactive-Visualization-Engine)', 'HTML5 \u002F Javascript', 'Yes'],\n['[HiGlass](https:\u002F\u002Fhiglass.io\u002F)', 'No', 'Chromatin interaction', '[Yes](https:\u002F\u002Fgithub.com\u002Fhiglass)', 'Python \u002F Javascript', 'No'],\n['[IGV](https:\u002F\u002Figv.org\u002F)','No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002Figvteam)', 'Java \u002F Javascript', 'Yes'],\n['[JBrowse](https:\u002F\u002Fjbrowse.org\u002Fjb2\u002F)', 'No', 'General', '[Yes](https:\u002F\u002Fjbrowse.org\u002Fjb2\u002Fdownload\u002F)', 'Javascript', 'Yes'],\n['[NGB](https:\u002F\u002Flifescience.opensource.epam.com\u002Fngb\u002Findex.html)','No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002Fepam\u002FNGB)', 'Java \u002F Javascript', 'No'],\n['[pyGenomeTracks](https:\u002F\u002Fpygenometracks.readthedocs.io\u002Fen\u002Flatest\u002F)','No', 'General', '[Yes](https:\u002F\u002Fgithub.com\u002Fdeeptools\u002FpyGenomeTracks)', 'Python', 'Yes'],\n['[Web Apollo](http:\u002F\u002Fgenomearchitect.org\u002F)', 'No', 'Collaborative genome annotation', '[Yes](https:\u002F\u002Fgithub.com\u002FGMOD\u002FApollo)', 'Groovy \u002F Java \u002F Javascript', 'No'],\n]",":table-number":P,":table-caption":_},children:[{type:a,value:Q}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"For more complete lists of genome browsers, please visit "},{type:b,tag:e,props:{href:"https:\u002F\u002Fbio.tools\u002Ft?page=1&q=%27Genome%20browser%27&sort=citationDate&ord=desc",rel:[g,h,i],target:j},children:[{type:a,value:"bio.tools"}]},{type:a,value:"\nor the "},{type:b,tag:e,props:{href:"https:\u002F\u002Fcmdcolin.github.io\u002Fawesome-genome-visualization",rel:[g,h,i],target:j},children:[{type:a,value:"awesome genome visualization"}]},{type:a,value:" web page (genome browser\nsoftware only)."}]},{type:a,value:c},{type:b,tag:W,props:{":quote":"\"The FAIRtracks team advocate the use of the term \\\"track\\\" also in analysis scenarios outside the realm of genome visualisation. \""},children:[{type:a,value:Q}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The term \"track\" implies a mental visual model of elements located along some line. In the domain of genomics\nthis typically relates to a coordinate system defined by a reference genome sequence collection.\nThis is a crucial model to keep in mind also for non-visual analysis scenarios. There exists, to our knowledge, no\nother term in broad use which is both general and precise enough to cover the same heterogeneity of data\ncontent, data representation and analysis scenarios as the term \"track\" (or a variation\nthereof). Hence, the FAIRtracks team advocate the use of the term \"track\" also in analysis scenarios\noutside the realm of genome visualization."}]}]},dir:v,path:"\u002Ftopics\u002Fs02-genome-browsers",extension:w,createdAt:l,updatedAt:l},{slug:"s03-track-types",title:"Track Types",img:$,caption:"Geometrical delineation of genomic tracks into 15 track types [(Gundersen, S *et al.*, 2011)](https:\u002F\u002Fdoi.org\u002F10.1186\u002F1471-2105-12-494)",ingress:"Track data can be categorized into 15 theoretical track types",toc:[],body:{type:u,children:[{type:b,tag:d,props:{},children:[{type:a,value:"As exemplified in the section "},{type:b,tag:e,props:{href:"#s02-genomic-tracks"},children:[{type:a,value:V}]},{type:a,value:", a multitude of different types of genome-wide\ndata, experimental and otherwise, can be condensed in the form of track files. The variety of data that can be\nexpressed in track form has consequences for the representation of the data, both for mathematical modeling\nand for the file formats used for storage and exchange."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Gundersen, S "},{type:b,tag:q,props:{},children:[{type:a,value:aa}]},{type:a,value:ab},{type:b,tag:e,props:{href:ac,rel:[g,h,i],target:j},children:[{type:a,value:ad}]},{type:a,value:" reviewed and categorized the landscape of\ntrack data representation. This paper proposed a unified conceptual framework which delineated the\nmodel space into 15 possible mathematical\u002Fgeometrical models, named "},{type:b,tag:k,props:{},children:[{type:a,value:ae}]},{type:a,value:". These 15 models can be used to\nrepresent, visualize, and analyze fundamentally different types of track data."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Logically, these "},{type:b,tag:k,props:{},children:[{type:a,value:ae}]},{type:a,value:" correspond to whether four different core properties of the data are present:\n"},{type:b,tag:k,props:{},children:[{type:a,value:R}]},{type:a,value:af},{type:b,tag:k,props:{},children:[{type:a,value:H}]},{type:a,value:af},{type:b,tag:k,props:{},children:[{type:a,value:ag}]},{type:a,value:", and "},{type:b,tag:k,props:{},children:[{type:a,value:S}]},{type:a,value:". Figure 3 illustrates the similarities and\ndifferences of these 15\ndifferent types of track data. If we look closely at the fifteen combinations, starting at the top left of Figure 3\nand moving downwards, an interesting pattern appears. From Gundersen, S "},{type:b,tag:q,props:{},children:[{type:a,value:aa}]},{type:a,value:",\n"},{type:b,tag:e,props:{href:ac,rel:[g,h,i],target:j},children:[{type:a,value:ad}]},{type:a,value:ab},{type:b,tag:"sup",props:{id:"fnref-1"},children:[{type:b,tag:e,props:{href:"#fn-1",className:["footnote-ref"]},children:[{type:a,value:P}]}]},{type:a,value:":"}]},{type:a,value:c},{type:b,tag:"blockquote",props:{},children:[{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"We start at the base case where the only core informational property is the "},{type:b,tag:k,props:{},children:[{type:a,value:R}]},{type:a,value:" between the track elements. In\nthis case, each track element represents an exact base pair location on the genome, denoting "},{type:b,tag:q,props:{},children:[{type:a,value:"e.g. viral insertion\nsites"}]},{type:a,value:". We call this track type "},{type:b,tag:k,props:{},children:[{type:a,value:"Points (P)"}]},{type:a,value:". Adding informative values to this case, "},{type:b,tag:q,props:{},children:[{type:a,value:"e.g. associating SNPs with\nallele frequencies"}]},{type:a,value:", we get the track type "},{type:b,tag:k,props:{},children:[{type:a,value:"Valued Points (VP)"}]},{type:a,value:". In\nthe next two cases, the property "},{type:b,tag:k,props:{},children:[{type:a,value:H}]},{type:a,value:" is added, resulting in the track types "},{type:b,tag:k,props:{},children:[{type:a,value:"Segments (S)"}]},{type:a,value:I},{type:b,tag:k,props:{},children:[{type:a,value:"Valued\nSegments (VS)"}]},{type:a,value:". Segments are probably the most common track type of existing tracks, representing "},{type:b,tag:q,props:{},children:[{type:a,value:"e.g. genes or\nexons, TF binding sites, or DNA methylation regions"}]},{type:a,value:". Valued segments could denote "},{type:b,tag:q,props:{},children:[{type:a,value:"e.g. genes with\nassociated expression levels"}]},{type:a,value:A}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Moving on, we remove the "},{type:b,tag:k,props:{},children:[{type:a,value:ag}]},{type:a,value:I},{type:b,tag:k,props:{},children:[{type:a,value:R}]},{type:a,value:" properties, leaving only "},{type:b,tag:k,props:{},children:[{type:a,value:H}]},{type:a,value:". Such tracks consist of\nsegments covering all base pairs of the genome, i.e. a partition of the genome into potentially unequal pieces.\nHence, the track type is called "},{type:b,tag:k,props:{},children:[{type:a,value:"Genome Partition (GP)"}]},{type:a,value:". Examples of this track type include the "},{type:b,tag:q,props:{},children:[{type:a,value:"partition\nof a genome into chromosomes or predicted chromatin states"}]},{type:a,value:". Adding a "},{type:b,tag:k,props:{},children:[{type:a,value:"value"}]},{type:a,value:" to each part of a partition creates a\nStep Function (SF), covering the whole genome with values. Examples of such tracks include "},{type:b,tag:q,props:{},children:[{type:a,value:"mapped sequence read\ncounts (e.g. a BAM file), or the signal files that are typically generated in a ChIP-seq workflow"}]},{type:a,value:". Removing the\n"},{type:b,tag:k,props:{},children:[{type:a,value:H}]},{type:a,value:" core property, the step function track is transformed into a track of type Function (F), where every\nbase pair has an associated value. Examples of function tracks are tracks with close dependency on the genome\nsequence, such as "},{type:b,tag:q,props:{},children:[{type:a,value:"GC content tracks, or the actual genome sequence itself"}]},{type:a,value:". We call the seven track types outlined\nhere for "},{type:b,tag:k,props:{},children:[{type:a,value:"basic track types"}]},{type:a,value:A}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The fourth core informational property, "},{type:b,tag:k,props:{},children:[{type:a,value:S}]},{type:a,value:", can be envisioned as an orthogonal extension to the\nprevious discussion. Adding interconnections, or edges, to the seven track types previously outlined (first column\nof Figure 3) defines linked versions of the same track types, e.g. "},{type:b,tag:k,props:{},children:[{type:a,value:"Linked Segments (LS)"}]},{type:a,value:" or "},{type:b,tag:k,props:{},children:[{type:a,value:"Linked Step Function\n(LSF)"}]},{type:a,value:" (second column of Figure 3). Examples of linked track dataset include "},{type:b,tag:q,props:{},children:[{type:a,value:"ChIA-PET chromatin interactions\n("},{type:b,tag:k,props:{},children:[{type:a,value:"Linked Segments, LS"}]},{type:a,value:") and Hi-C contact maps ("},{type:b,tag:k,props:{},children:[{type:a,value:"Linked Genome Partition, LGP"}]},{type:a,value:")"}]},{type:a,value:A}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"[...]"}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"To complete the picture, a last track type needs to be defined. If only the "},{type:b,tag:k,props:{},children:[{type:a,value:S}]},{type:a,value:" core property is\ndefined, track elements do not have gaps between them, lengths, or values. [...] The linked\nbase pairs track type is mostly suggestive at this point. [...] We refer to the eight linked track types as the\n"},{type:b,tag:k,props:{},children:[{type:a,value:"extended track types"}]},{type:a,value:A}]},{type:a,value:c}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The track type concept is useful not only for categorizing tracked data itself but also for a\ncategorizing possible analyses in terms of the track types of their input and output files. For example, "},{type:b,tag:q,props:{},children:[{type:a,value:"ChIP-seq peak\ncalling"}]},{type:a,value:" can be viewed as a data reduction operation, transforming input track files of type "},{type:b,tag:k,props:{},children:[{type:a,value:"Step Function (SF)"}]},{type:a,value:"\n("},{type:b,tag:q,props:{},children:[{type:a,value:"i.e. sequence read files"}]},{type:a,value:") into output track files of type "},{type:b,tag:k,props:{},children:[{type:a,value:"Valued Segments (VS)"}]},{type:a,value:", where the values represent the\nintensities of the peaks that are called."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Note that all 15 track types are supported by the FAIRTracks metadata standard, as well as the GTrack and GSuite file\nformats, which are all described in the "},{type:b,tag:T,props:{to:"\u002Fstandards"},children:[{type:a,value:"Standards"}]},{type:a,value:" page."}]},{type:a,value:c},{type:b,tag:"div",props:{className:["footnotes"]},children:[{type:a,value:c},{type:b,tag:"hr",props:{},children:[]},{type:a,value:c},{type:b,tag:ah,props:{},children:[{type:a,value:c},{type:b,tag:p,props:{id:"fn-1"},children:[{type:a,value:"The quote is slightly edited. Text formatting have been added and some examples have been substituted by more recent ones."},{type:b,tag:e,props:{href:"#fnref-1",className:["footnote-backref"]},children:[{type:a,value:"↩"}]}]},{type:a,value:c}]},{type:a,value:c}]}]},dir:v,path:"\u002Ftopics\u002Fs03-track-types",extension:w,createdAt:l,updatedAt:l},{slug:"s04-finding-tracks",title:"Finding Tracks",img:E,caption:t,ingress:t,toc:[],body:{type:u,children:[{type:b,tag:d,props:{},children:[{type:a,value:"The FAIRtracks project maintains a draft standard for metadata about genomic track datasets. The aim is that the\nstandard will function as a minimal metadata exchange standard for harmonization of\nheterogeneous track metadata available in various [track collections or data portals, as described below]\n(\u002Fs05-track-collections). Once a process has been set up to transform existing or novel track collection to follow\nthe FAIRtracks standard (which in most cases also includes aspects of [FAIRification]\n(\u002Fs07-fairification)), the harmonized metadata can be imported into a central service for discovery of track files\nacross the various track collections that are integrated, named "},{type:b,tag:T,props:{to:"\u002Fservices#trackfind"},children:[{type:a,value:"TrackFind"}]},{type:a,value:A}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"TrackFind provides both a web user interface as well a REST API to allow access by downstream tools and scripts.\nIn the web portal, TrackFind provides a hierarchical browser of the various metadata fields mapped to the standard\nincluding summary lists of the different values that are represented in the indexed metadata. Through the graphical user\ninterface, these values can be used to define categorical search queries (in the form of a SQL query). The results\nof the queries are available for browsing or downloadable as JSON or GSuite formats."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The following screencast demonstrates use of track find to locate set of track files from the BLUEPRINT track\ncollection:"}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The FAIRtracks metadata standard is a minimal standard, it is designed to be interoperable with other resources and\nserviced. One example of such interoperability is the ability to launch external websites describing particular\nvocabulary terms or records of source material in other repositories. For more about this, see sections on\n"},{type:b,tag:T,props:{to:"\u002Fs08-identifiers"},children:[{type:a,value:ai}]},{type:a,value:I},{type:b,tag:e,props:{href:aj},children:[{type:a,value:ak}]},{type:a,value:A}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Integrating with TrackFind. A key feature of TrackFind is the ability to integrates search functionality in other\nsoftware frameworks and tools. Currently this integration can be implemented by making use of the rest API,\ndocumented here. We aim to develop libraries in common premium language is that rap this API to simplify use an\nintegration,. Any contributions to this and is highly appreciated in. Currently, we have integrated TrackFind with\nthe GSuite HyperBrowser and EPICO.\nThe track find client tool in the dynamic have a browser illustrates a powerful feature of the TrackFind API: when\nspecifying the meta-data filled the criteria, the user will only be presented values that are available given the\nprevious filters that has been selected. Hence, the user will be guided interactively at every step in the\nfiltering process, reducing the probability for empty search results. For example, the user can start off by\nselecting transcription facto chipset As the experiment type, alongside a set of Target transcription factors, and\nwill then be presented with a list of Sample types (cell types, cell lines, etc.), where the TF‘s question I have\nbeen Investigated. This interactive reduction in selectable values can be extended to any metadata fields, e.g.,\nprojects information, file formats, Data usage limitations."}]}]},dir:v,path:"\u002Ftopics\u002Fs04-finding-tracks",extension:w,createdAt:l,updatedAt:l},{slug:"s05-track-collections",title:"Track Collections",img:al,caption:"Most recommanded standard recommended by policies [(fairsharing.org)](https:\u002F\u002Ffairsharing.org\u002Fsummary-statistics)",ingress:"Access genomic tracks using Track collection",toc:[],body:{type:u,children:[{type:b,tag:d,props:{},children:[{type:a,value:"Track collections are structures that group one or more types of resources such as databases, standards or policies.\nThey can represent a domain, a project or an organization. Deposition of data relies on multiple standards for\ntracks deposit; consequently, available data and metadata may differ from collection to collection."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Many efforts have been made to uniformize these standards to follow the FAIR data principles. For example, the\nMinimum Information About a Microarray Experiment (MIAME) standard has been adopted by the most popular track\ncollections such as "},{type:b,tag:e,props:{href:"https:\u002F\u002Fwww.ebi.ac.uk\u002Farrayexpress\u002F",rel:[g,h,i],target:j},children:[{type:a,value:"ArrayExpress"}]},{type:a,value:" or [Gene Expression Omnibus]("},{type:b,tag:e,props:{href:am,rel:[g,h,i],target:j},children:[{type:a,value:am}]},{type:a,value:".\nncbi.nlm.nih.gov\u002Fgeo\u002F) (GEO). Another example is regarding Novel DNA sequence, novel RNA sequence, and novel genome\nassembly data that must be deposited to repositories that are part of the International Nucleotide Sequence\nCollaboration (INSDC) following a standard feature table. Consequently, tracks can be featured on multiple\ncollections with or without a cross identifier."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Projects exist to help find a relevant collection to one's interest such as [Repository Finder]\n("},{type:b,tag:e,props:{href:an,rel:[g,h,i],target:j},children:[{type:a,value:an}]},{type:a,value:") and "},{type:b,tag:e,props:{href:"https:\u002F\u002Ffairsharing.org",rel:[g,h,i],target:j},children:[{type:a,value:"FAIRsharing.org"}]},{type:a,value:". Moreover, Track hubs are\ncreated by researchers or projects to easily visualize genomic tracks in popular genome browsers such as [UCSC] or\nEnsembl and can be found on [UCSC Track hub list]("},{type:b,tag:e,props:{href:ao,rel:[g,h,i],target:j},children:[{type:a,value:ao}]},{type:a,value:"? redirect=manual&source=genome.ucsc.edu)\nor "},{type:b,tag:e,props:{href:"https:\u002F\u002Fwww.trackhubregistry.org",rel:[g,h,i],target:j},children:[{type:a,value:"Track Hub registtry"}]},{type:a,value:A}]},{type:a,value:c},{type:b,tag:Z,props:{":headers":"['Database name', 'Scientific domain', 'Organism(s)', 'Assay type(s)', 'Organisation', 'Notes']",":items":"[\n['[Array Express](https:\u002F\u002Fwww.ebi.ac.uk\u002Farrayexpress\u002F)','Generic data warehouse','Multiple','Multiple','EMBL-EBI',''],\n['[Gene Expression Omnibus (GEO)](http:\u002F\u002Fwww.ncbi.nlm.nih.gov\u002Fgeo\u002F)','Generic data warehouse','Multiple','Multiple','National Center for Biotechnology Information',''],\n['[ENCODE](https:\u002F\u002Fwww.encodeproject.org)','Epigenomics','H. sapiens, M. musculus','DNA methylation \u002F histone modification \u002F chromatin accessibility \u002F transcription factor binding \u002F RNA-binding proteins \u002F  mRNA profiling \u002F ncRNA expression profiling \u002F 3D chromatin structure \u002F DNA replication timing','ENCODE consortium','Contain Roadmap and ModENCODE data, data in Array express, GEO and IHEC'],\n['[Roadmap Epigenomics](http:\u002F\u002Fwww.roadmapepigenomics.org)','Epigenomics \u002F Stem cells \u002F primary ex vivo tissues','H. sapiens','DNA methylation \u002F histone modifications \u002F chromatin accessibility \u002F small RNA transcripts','Roadmap Epigenomics Mapping Consortium (NIH)',''],\n['[ModENCODE](http:\u002F\u002Fwww.modencode.org)','Epigenomics \u002F Genomics','C. elegans, D. melanogaster','gene structure \u002F mRNA profiling \u002F ncRNA expression profiling \u002F transcription factor binding \u002F histone modifications \u002F histone replacement \u002F chromatin structure \u002F DNA replication initiation \u002F DNA replication timing \u002F copy number variation','ModEncode Consortium  (NIH)',''],\n['[Blueprint](https:\u002F\u002Fwww.blueprint-epigenome.eu)','Epigenomics \u002F Haematopoietic cells','H. sapiens, M. musculus','DNA methylation \u002F transcription factor binding \u002F chromatin accessibility \u002F mRNA profiling','BLUEPRINT Consortium  (IHEC)','Data in EGA'],\n['[FANTOM](https:\u002F\u002Ffantom.gsc.riken.jp)','Epigenomics','H. sapiens, M. musculus','mRNA profiling \u002F ncRNA expression profiling','Functional ANnoTation Of the Mammalian genome Consortium',''],\n['[International Human Epigenome Consortium (IHEC)](https:\u002F\u002Fepigenomesportal.ca)','Epigenomics','H. sapiens, M. musculus','DNA methylation \u002F histone modifications \u002F mRNA profiling','International Human Epigenome Consortium (IHEC)','Contain Blueprint, ENCODE data'],\n['[Cancer Genome Atlas Program (TCGA)](https:\u002F\u002Fportal.gdc.cancer.gov)','Epigenomics \u002F Genomics \u002F Cancer','H. sapiens','clinical data \u002F copy number variation \u002F single nucleotide variation \u002F DNA sequence variation \u002F imaging \u002F DNA methylation \u002F miRNA expression profiling \u002F mRNA expression profiling \u002F protein expression profiling','NCI (NIH) and the National Human Genome Research','Portal with CPTAC, MMRF, CMI, TARGET, HCMI and CGCI data'],\n['[International Cancer Genome Consortium (ICGC)](https:\u002F\u002Fdcc.icgc.org)','Epigenomics \u002F Genomics \u002F Cancer','H. sapiens','clinical data \u002F DNA methylation \u002F miRNA expression profiling \u002F mRNA expression profiling \u002F DNA sequence variation \u002F copy number variation','NCI (NIH) and the International Cancer Genome Consortium','Contain PCAWG data'],\n['[Pan-Cancer Analysis of Whole Genomes (PCAWG)](https:\u002F\u002Fdcc.icgc.org\u002Fpcawg\u002F)','Genomics \u002F Cancer','H. sapiens','clinical data \u002F copy number variation \u002F single nucleotide variation \u002F DNA sequence variation','Pan-Cancer Analysis of Whole Genomes consortium',''],\n['[DECIPHER](https:\u002F\u002Fwww.deciphergenomics.org)','Genomics \u002F Rare disease','H. sapiens','clinical data \u002F copy number variation \u002F DNA sequence variation','Sanger Institute',''],\n['[GTEx Portal](https:\u002F\u002Fgtexportal.org)','Epigenomics \u002F Genomics','H. sapiens','clinical data \u002F snRNA expression profiling \u002F mRNA expression profiling \u002F DNA sequence variation \u002F single nucleotide variation \u002F eQTL','The Genotype-Tissue Expression consortium',''],\n['[GWAS Catalog](https:\u002F\u002Fwww.ebi.ac.uk\u002Fgwas\u002F)','Genomics','H. sapiens','single nucleotide variation','National Human Genome Research Institute (NIH) and European Bioinformatics Institute',''],\n['[RNACentral](https:\u002F\u002Frnacentral.org)','Epigenomics','H. sapiens, M. musculus, A. thaliana, C. elegans, D. rerio, D. melanogaster. S. pombe','ncRNA expression profiling','European Bioinformatics Institute','Integration access of multiple databases \\*'],\n['[MirBase](https:\u002F\u002Fwww.mirbase.org)','Epigenomics','Multiple (Alveolata, Metazoa, Mycetozoa, Viridiplantae, Viruses)','miRNA expression profiling','University of Manchester',''],\n['[MirGeneDB](https:\u002F\u002Fmirgenedb.org)','Epigenomics','Multiple (Metazoa)','miRNA expression profile','RNAcentral group',''],\n['[dbSNP](https:\u002F\u002Fwww.ncbi.nlm.nih.gov\u002Fsnp\u002F)','Genomics','Multiple','single nucleotide variation','National Center for Biotechnology Information (NIH)',''],\n['[UniBind](https:\u002F\u002Funibind.uio.no)','Epigenomics','A. thaliana, C. elegans, D. rerio, D. melanogaster, H. sapiens, M. musculus, R. novergicus, S. cerevisiae, S. pombe ','transcription factor binding','Unibind group',''],\n['[ReMap](https:\u002F\u002Fremap.univ-amu.fr)','Epigenomics','H. sapiens, M. musculus, D. melanogaster, A. thaliana','transcription factor binding','ReMap group','Reprocessed data from GEO and ENCODE'],\n['[GTRD](http:\u002F\u002Fgtrd.biouml.org)','Epigenomics','A. thaliana, C. elegans, D. rerio, D. melanogaster, G. gallus, H. sapiens, M. musculus, R. novergicus, S. cerevisiae, S. pombe','transcription factor binding \u002F chromatin accessibility \u002F RNA initiation','BioUML',''],\n['[GeneBank](https:\u002F\u002Fwww.ncbi.nlm.nih.gov\u002Fgenbank\u002F)\u002F[DDBJ](https:\u002F\u002Fwww.ddbj.nig.ac.jp\u002Fddbj\u002F)\u002F[ENA](http:\u002F\u002Fwww.ebi.ac.uk\u002Fena\u002F)','Genomics','Multiple','DNA sequence','National Center for Biotechnology Information (NIH) \u002F DNA DataBank of Japan \u002F European Bioinformatics Institute',],\n['[Expression Atlas](https:\u002F\u002Fwww.ebi.ac.uk\u002Fgxa\u002F)','Genomics','Multiple','RNA profile expression','European Bioinformatics Institute','Reprocess data from COVID-19 Data portal, ENCODE, BLUEPRINT, FANTOM, the Human Atlas Project, Cancer Cell Line Encyclopedia, HipSci, GTEx, PCAWG, Genomics of Drug Sensitivity in Cancer Project, HDBR, Gramene'],\n['[FlyBase](http:\u002F\u002Fflybase.org)','Genomics \u002F Epigenomics','D. simulans, D. sechellia, D. melanogaster, D. yakuba, D. erecta, D. ananassae, D. pseudoobscura, D. persimillis, D. willistoni, D. mojavensis, D. virillis, D. grimshawi','DNA sequence \u002F DNA sequence variation \u002F mRNA profile expression \u002F ncRNA profile expression \u002F 3D chromatin structure \u002F protein sequence','FlyBase consortium ','Data in RNA central, part of [Alliance of Genome Resource](https:\u002F\u002Fwww.alliancegenome.org)'],\n['[Mouse Genome Informatics (MGI)](http:\u002F\u002Fwww.informatics.jax.org )','Genomics \u002F Epigenomics','M. musculus','DNA sequence \u002F RNA expression profile','MGI project','Data in RNA central, part of [Alliance of Genome Resource](https:\u002F\u002Fwww.alliancegenome.org)'],\n['[Rat Genome Database (RGD)](https:\u002F\u002Frgd.mcw.edu)','Genomics \u002F Epigenomics',' R. norvegicus, M. musculus, H. sapiens, C. lanigera, P. paniscusm, C. familiaris, I. tridecemlineatus, S. scrofa, H. glaber, C. sabaeus','DNA sequence \u002F RNA expression profile','RGD group (NIH)','Data in RNA central, part of [Alliance of Genome Resource](https:\u002F\u002Fwww.alliancegenome.org)'],\n['[WormBase](https:\u002F\u002Fwormbase.org)','Genomics \u002F Epigenomics','C. elegans and related nematodes','DNA sequence \u002F RNA expression profile','WormBase consortium','Data in RNA central, part of [Alliance of Genome Resource](https:\u002F\u002Fwww.alliancegenome.org)'],\n['[ZFIN](http:\u002F\u002Fzfin.org )','Genomics \u002F Epigenomics','D. rerio','DNA sequence \u002F RNA expression profile','Zebrafish Information Network group','Data in RNA central, part of [Alliance of Genome Resource](https:\u002F\u002Fwww.alliancegenome.org)'],\n['[Saccharomyces Genome Database (SGD)](https:\u002F\u002Fyeastgenome.org)','Genomics \u002F Epigenomics','S. cerevisiae','DNA sequence \u002F RNA expression profile','SGD group','Data in RNA central, part of [Alliance of Genome Resource](https:\u002F\u002Fwww.alliancegenome.org)'],\n['[Xenbase](http:\u002F\u002Fwww.xenbase.org\u002F)','Genomics \u002F Epigenomics','X. laevis, X. tropicalis','DNA sequence \u002F RNA expression profile \u002F protein sequence \u002F tanscription sequence binding','Xenbase group',],\n['[The Arabidopsis Information Resource (TAIR)](https:\u002F\u002Fwww.arabidopsis.org )','Genomics','A. thaliana','DNA sequence \u002F RNA expression profile','TAIR group','Data in RNA central'],\n['[Influenza Research Database (IRD)](https:\u002F\u002Fwww.fludb.org)','Genomics \u002F Disease','Influenza virus, hosts','DNA sequence \u002F 3D protein structure','IRD group',''],\n['[VEuPathDB](https:\u002F\u002Fvectorbase.org)','Genomics \u002F Disease','Eukaryotic pathogens and invertebrate vectors of disease','DNA sequence','Eukaryotic Pathogen, Vector and Host Informatics Resource',''],\n]",":table-number":P,":table-caption":_},children:[{type:a,value:Q}]}]},dir:v,path:"\u002Ftopics\u002Fs05-track-collections",extension:w,createdAt:l,updatedAt:l},{slug:"s06-analysing-tracks",title:"Analysing Tracks",img:E,caption:t,ingress:t,toc:[],body:{type:u,children:[{type:b,tag:d,props:{},children:[{type:a,value:t}]}]},dir:v,path:"\u002Ftopics\u002Fs06-analysing-tracks",extension:w,createdAt:l,updatedAt:l},{slug:"s07-fairification",title:"FAIRification of Track Metadata",img:E,caption:t,ingress:t,toc:[],body:{type:u,children:[{type:b,tag:d,props:{},children:[{type:a,value:t}]}]},dir:v,path:"\u002Ftopics\u002Fs07-fairification",extension:w,createdAt:l,updatedAt:l},{slug:"s08-identifiers",title:ai,img:ap,caption:"Assign identify content and provide a persistent link to the location of your track data",ingress:"Assign a persistent reference to your published track data",toc:[],body:{type:u,children:[{type:b,tag:d,props:{},children:[{type:a,value:"A Globally Unique Identifier (GUID), also referred to as Universal Unique Identifier (UUID),\nis a 128-bit reference number that identifies unique Internet objects or data.\nA GUID is highly unlikely to repeat when generated despite the absence of a central authority to ensure uniqueness.\nThere are currently a few instances of GUIDs being assigned to track files or associated data.\nIn many cases, only the raw sequence files are assigned identifiers e.g. the accession number to data repositories such as the\n"},{type:b,tag:e,props:{href:"https:\u002F\u002Fwww.ncbi.nlm.nih.gov\u002Fsra",rel:[g,h,i],target:j},children:[{type:a,value:"Sequence Read Archive (SRA)"}]},{type:a,value:" or the "},{type:b,tag:e,props:{href:"https:\u002F\u002Fega-archive.org\u002F",rel:[g,h,i],target:j},children:[{type:a,value:"European Genome-Phenome Archive (EGA)"}]},{type:a,value:".\nThe "},{type:b,tag:e,props:{href:"https:\u002F\u002Fwww.encodeproject.org\u002F",rel:[g,h,i],target:j},children:[{type:a,value:"ENCODE project"}]},{type:a,value:" represents a notable exception:\neach track is associated with an identifier resolvable through "},{type:b,tag:e,props:{href:aq,rel:[g,h,i],target:j},children:[{type:a,value:ar}]},{type:a,value:"\nand an associated web page.\nHowever, a universally accessible service to assign and register identifiers to single track files and collections is currently missing.\nTrack files contain condensed data from bioinformatics workflows and are thus prone to contain biases due to specific parameter settings.\nWe therefore strongly recommend the implementation of a track registry that preserves the full context by assigning global identifiers not only to the track files\nbut also to the associated metadata. The FAIRtracks draft standard is developed with this concept in mind and could be suitable for use as a basis for a registry of track metadata."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The current FAIRtracks implementation does not require the inclusion of GUID to track data files, due to the lack of a global repository.\nWe require instead the inclusion of Uniform Resource Locators (URLs) to track files,\nwhich, however, come without any guarantees of persistence or uniqueness.\nIn case a direct identifier is not attached to a track file, the identifier of a parent record (e.g. study or experiment) can be used instead.\nOn top of this, FAIRtracks requires a global identifier for the metadata file itself using a document identifier (DOI).\nIn principle, a track file can be uniquely pin-pointed by a joint identifier containing the DOI of the FAIRtracks document and the track URL."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"On top of a DOI associated with a FAIRtrakcs document, our policy also requires support for DOI versioning and DOI reservation prior to publication.\nWe currently recommend "},{type:b,tag:e,props:{href:"https:\u002F\u002Fzenodo.org\u002F",rel:[g,h,i],target:j},children:[{type:a,value:"Zenodo"}]},{type:a,value:", as it supports these features, but we would extend our list\nof recommended repositories and archives to any domain-specific service meeting our requirements.\nThe capability of identifying specific collections would also improve the provenance of analysis making use of track data.\nThis could be implemented by only allowing a single track collection in a FAIRtracks document and then referring to the DOI.\nThere is, however, an issue with granularity, as the number of published datasets, in this case, would be impractically high,\nas no publishing platform supports identifiers at a sub-dataset level in combination with versioning and pre-reservation of DOIs.\nCreating a separate service to assign and register identifiers at the track collection level seems a better solution."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"FAIRtracks supports and recommends the inclusion of global identifiers to external records containing detailed metadata.\nWe require these global identifiers represented in Compact Uniform Resource Identifies (CURIE) form resolvable through "},{type:b,tag:e,props:{href:aq,rel:[g,h,i],target:j},children:[{type:a,value:ar}]},{type:a,value:".\nA mapping service from existing URIs to the corresponding CURIEs is desirable, as it would enhance the conversion of existing metadata to the FAIRtracks standard."}]}]},dir:v,path:"\u002Ftopics\u002Fs08-identifiers",extension:w,createdAt:l,updatedAt:l},{slug:aj,title:ak,img:as,caption:"Phagocyte definition according to [Cell Ontology](http:\u002F\u002Fwww.ontobee.org\u002Fontology\u002FCL?iri=http:\u002F\u002Fpurl.obolibrary.org\u002Fobo\u002FCL_0000234).",ingress:"Biological terms used to describe metadata must be formalised",toc:[],body:{type:u,children:[{type:b,tag:d,props:{},children:[{type:a,value:"Ontologies are a formal way of representing knowledge in which concepts are described by their meaning and their\nrelationship. Ontologies assign to each concept (or term) its own unique identifier (ID), thus facilitating\narchiving, storing and accessing data in databases."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"For example, a phagocyte is a cell type represented by a term (or concept) with a description, an ID and\nrelationships. Its relationship defined it as a subclass of Native cell, Motile cell, stuff Accumulating cell, and\ndefensive cell; other terms with their own definition, ID and relationships (see diagram). A Cell Ontology allows\nthe definition of all of those concepts."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Interoperability across ontologies is invaluable to describe complex biological knowledge with composite\nannotations. Ontologies attribution of unique IDs to biological terms not only facilitates storing and searching\ndata in databases but also linking information between them. Using the previous example, the phagocyte cell type as\na unique ID in "},{type:b,tag:e,props:{href:"https:\u002F\u002Fobofoundry.org\u002Fontology\u002Fcl.html",rel:[g,h,i],target:j},children:[{type:a,value:"Cell Ontology"}]},{type:a,value:", but also foreign ID linking it to the\n"},{type:b,tag:e,props:{href:"http:\u002F\u002Fgeneontology.org",rel:[g,h,i],target:j},children:[{type:a,value:"Gene Ontology"}]},{type:a,value:" describing phagocytosis as a biological process linked to the cell."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"To support interoperability, "},{type:b,tag:e,props:{href:"https:\u002F\u002Fobofoundry.org",rel:[g,h,i],target:j},children:[{type:a,value:"OBO foundry"}]},{type:a,value:" is a community whose goal is to provide a set of\ninteroperable ontologies that are both logically well-formed and scientifically accurate. They review participating\nontologies according to sets of "},{type:b,tag:e,props:{href:"https:\u002F\u002Fobofoundry.org\u002Fprinciples\u002Ffp-000-summary.html",rel:[g,h,i],target:j},children:[{type:a,value:"principles and frameworks"}]},{type:a,value:".\n"},{type:b,tag:e,props:{href:"https:\u002F\u002Fbioportal.bioontology.org",rel:[g,h,i],target:j},children:[{type:a,value:"BioPortal"}]},{type:a,value:" proposes a service to access OBO foundry's complying ontologies."}]}]},dir:v,path:"\u002Ftopics\u002Fs09-ontologies",extension:w,createdAt:l,updatedAt:l},{slug:"s10-validation",title:"Validation and Augmentation",img:at,caption:"Validate and augment a FAIRtracks document",ingress:"Validate your metadata document and augment it automatically by fetching human-readable entries",toc:[],body:{type:u,children:[{type:b,tag:d,props:{},children:[{type:a,value:"We developed the FAIRtracks standard and the associated ecosystem with the aim of fulfilling two main objectives:"}]},{type:a,value:c},{type:b,tag:ah,props:{},children:[{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Facilitating the deposition of track data by defining a minimal set of metadata fields to fill out to be compliant with the FAIR data principles."}]},{type:a,value:c}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Improving downstream usage of track data through more standardized retrieval and extraction operations enabled by a well-defined metadata format."}]},{type:a,value:c}]},{type:a,value:c}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Let us consider the exemplary case of ontology terms. If data suppliers only provide identifiers, these would not be comprehensible to downstream\nhuman users and it would be extremely cumbersome to implement ontology lookup functionality in all the relevant tools and services.\nOn the other hand, forcing data suppliers to provide these services would discourage data deposition.\nThis clearly points out the need for an intermediate unique service providing ontology lookup functionality and other FAIR-oriented features."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The FAIRtracks ecosystem is created with the aim of filling this gap between (meta)data providers and consumers.\nOn top of the implementation of the FAIRtracks standard itself, this is enabled by metadata validation and augmentation services.\nWe provide the "},{type:b,tag:e,props:{href:"http:\u002F\u002Ffairtracks.bsc.es\u002Fapi\u002F",rel:[g,h,i],target:j},children:[{type:a,value:"FAIRtracks validation server"}]},{type:a,value:" as a RESTful API verifying the adherence of metadata lists to the\n"},{type:b,tag:e,props:{href:"https:\u002F\u002Fgithub.com\u002Ffairtracks\u002Ffairtracks_standard\u002F",rel:[g,h,i],target:j},children:[{type:a,value:"JSON Schema"}]},{type:a,value:" defining the FAIRtracks standard.\nThe server extends the standard JSON Schema validation technology through additional Python modules allowing for:"}]},{type:a,value:c},{type:b,tag:X,props:{},children:[{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"validation of ontology terms against specific ontology versions"}]},{type:a,value:c},{type:b,tag:p,props:{},children:[{type:a,value:"checking CURIEs against the registered entries at the "},{type:b,tag:e,props:{href:"identifiers.org"},children:[{type:a,value:"Identifiers.org resolution service"}]}]},{type:a,value:c}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The extended validator also supports document-set restrictions, like unique constraints enforcement and foreignProperty checks."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The "},{type:b,tag:e,props:{href:"https:\u002F\u002Ffairtracks.elixir.no\u002Fapi\u002F#api-Augmentation",rel:[g,h,i],target:j},children:[{type:a,value:"FAIRtracks metadata augmentation service"}]},{type:a,value:" is implemented as a RESTful API\ntaking a FAIRtracks-compliant JSON document as input and generating a set of additional human-readable fields containing\nontology labels, ontology versions, and more useful content for downstream users.\nSuch a service simplifies the medata entry on the providers' side and, at the same time,\nimproves the retrieval and extraction operations on the users' side in both the automated and manual usage scenarios.\nMoreover, the augmentation service can also be useful for integrating heterogeneous datasets."}]}]},dir:v,path:"\u002Ftopics\u002Fs10-validation",extension:w,createdAt:l,updatedAt:l},{slug:"s11-file_formats",title:"File Formats",img:E,caption:t,ingress:t,toc:[],body:{type:u,children:[{type:b,tag:d,props:{},children:[{type:a,value:"A track format aims to allow systematic handling of genomic data and metadata by standardizing information contained in files. A critical purpose of a track format is to be easily usable and sharable by humans and machines. In order to do that, they codify what minimal information must be present in a file and its formatting.\nGenerally, file formats are specific to distinct types of genomic information: genomic sequence, genomic variation, genomic expression, genomic ranges and genomic structure. A good overview of the most used data file format is available for data sharing is available at "},{type:b,tag:e,props:{href:"https:\u002F\u002Fgenome.ucsc.edu\u002FFAQ\u002FFAQformat.html#format1",rel:[g,h,i],target:j},children:[{type:a,value:"UCSC genome browser documentation"}]}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Although some consensus exists, a complete overview of all file formats is problematized by the number available. A wide variety of genomic information can be represented, from genomic sequences to complex three-dimensional structures (see Track Types topic), making the development of a single file format standard difficult. Currently, several formats exist, covering all types of genomic annotations, with some formats describing the same information with different standards. Moreover, variations do exist. For example, BED file formats have several variations, generally named BED X + Y, where X is the number of standard columns, and Y is the number of columns containing non-standard information. Those variations mean that files following the same file standard might not be compatible. It also adds to the plethora of already existing formats leads to a proliferation of formats that complicate interoperability and reusability of data."}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"File formats cover not only data and metadata present in a file but also computational file formats. Plain text format ensures human readability at the expense of file size. The quantity of information provided negatively impacts genomic track files size, complicating their programmatic manipulation. Several file formats have lossless compressed versions that are indexed binary variations such as BAM for SAM or BigBED for BED to address this issue. Several tools have been developed to convert and manipulate plain text and binary files interchangeably, such as "},{type:b,tag:e,props:{href:"https:\u002F\u002Fbedtools.readthedocs.io",rel:[g,h,i],target:j},children:[{type:a,value:"bedtools"}]},{type:a,value:I},{type:b,tag:e,props:{href:"http:\u002F\u002Fsamtools.sourceforge.net",rel:[g,h,i],target:j},children:[{type:a,value:"samtools"}]},{type:a,value:A}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"Standardization of genomic files format should be done at every step of a project and, ideally, be decided before data generation\u002Fgathering. This measure is even more critical for collaborative studies and projects where data generation and analysis is shared between different groups. International consortiums like "},{type:b,tag:e,props:{href:"https:\u002F\u002Fwww.encodeproject.org\u002Fhelp\u002Ffile-formats\u002F",rel:[g,h,i],target:j},children:[{type:a,value:"ENCODE"}]},{type:a,value:" and BLUEPRINT have dedicated part of their preliminary works to codify files formats for data and metadata. At a higher level, there is an international effort to select and standardize current formats to ensure the perenization of data and metadata. The "},{type:b,tag:e,props:{href:"https:\u002F\u002Fwww.ga4gh.org",rel:[g,h,i],target:j},children:[{type:a,value:"Global Alliance for Genomics and Health"}]},{type:a,value:" (GA4GH) is an alliance whose purpose is to \"cultivate a common framework of standards and harmonized approaches for effective and responsible genomic and health-related data sharing\". One of their work is to access and adopt open standards for genomic data sharing and provide guidance and tools to complete this goal ("},{type:b,tag:e,props:{href:au,rel:[g,h,i],target:j},children:[{type:a,value:au}]},{type:a,value:")."}]}]},dir:v,path:"\u002Ftopics\u002Fs11-file_formats",extension:w,createdAt:l,updatedAt:l},{slug:"s12-interoperability",title:"FAIR interoperability",img:E,caption:t,ingress:t,toc:[],body:{type:u,children:[{type:b,tag:d,props:{},children:[{type:a,value:t}]}]},dir:v,path:"\u002Ftopics\u002Fs12-interoperability",extension:w,createdAt:l,updatedAt:l}],imageAssetObjects:{"genomic-tracks.jpeg":{filename:U,isSvgImage:B,responsiveImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd56a02a-160.jpeg 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Faa55c3c-320.jpeg 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F101c100-640.jpeg 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1614d19-800.jpeg 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F4027e5b-960.jpeg 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F32f32a7-1280.jpeg 1280w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F385c3c6-1440.jpeg 1440w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F7a5e490-1600.jpeg 1600w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd3b3879-1920.jpeg 1920w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F78de58a-2560.jpeg 2560w",images:[{path:av,width:f,height:C},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Faa55c3c-320.jpeg",width:o,height:F},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F101c100-640.jpeg",width:m,height:J},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1614d19-800.jpeg",width:n,height:aw},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F4027e5b-960.jpeg",width:r,height:s},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F32f32a7-1280.jpeg",width:s,height:ax},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F385c3c6-1440.jpeg",width:y,height:x},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F7a5e490-1600.jpeg",width:z,height:ay},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd3b3879-1920.jpeg",width:x,height:G},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F78de58a-2560.jpeg",width:G,height:az}],src:av,width:f,height:C},responsiveWebpImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F78c3436-160.webp 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fe7b1492-320.webp 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd71c310-640.webp 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fcb8bb57-800.webp 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F835fea1-960.webp 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F65542f0-1280.webp 1280w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ffd96b23-1440.webp 1440w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F93a53c5-1600.webp 1600w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F17342e4-1920.webp 1920w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F4820a1f-2560.webp 2560w",images:[{path:aA,width:f,height:C},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fe7b1492-320.webp",width:o,height:F},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd71c310-640.webp",width:m,height:J},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fcb8bb57-800.webp",width:n,height:aw},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F835fea1-960.webp",width:r,height:s},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F65542f0-1280.webp",width:s,height:ax},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ffd96b23-1440.webp",width:y,height:x},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F93a53c5-1600.webp",width:z,height:ay},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F17342e4-1920.webp",width:x,height:G},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F4820a1f-2560.webp",width:G,height:az}],src:aA,width:f,height:C},optimizedImagePath:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F40fa760.jpeg",placeholderImagePath:Object("data:image\u002Fsvg+xml,%3Csvg xmlns='http:\u002F\u002Fwww.w3.org\u002F2000\u002Fsvg' viewBox='0 0 2655 3540'%3E%3Cfilter id='b'%3E%3CfeGaussianBlur stdDeviation='12' \u002F%3E%3C\u002Ffilter%3E%3Cpath fill='%23f3f3ee' d='M0 0h2655v3540H0z'\u002F%3E%3Cg filter='url(%23b)' transform='translate(7 7) scale(13.82813)' fill-opacity='.5'%3E%3Cpath fill='%230a0c00' d='M8 197h125v10H8z'\u002F%3E%3Cellipse fill='%23bfc562' rx='1' ry='1' transform='rotate(-119.1 94.4 -17.1) scale(41.99071 16.41534)'\u002F%3E%3Cellipse fill='%23525b00' cx='107' cy='114' rx='37' ry='4'\u002F%3E%3Cpath fill='%23493b34' d='M5.8 202.4l81.9-33.7L35 183l-22.4 8.5z'\u002F%3E%3Cpath fill='%23838b36' d='M130.2 152.7L79 150.2l38.1-6.3 57-8.4z'\u002F%3E%3Cpath fill='%237c9387' d='M172.5 231.5l-38-13.8L77 207.6l35.4-12.1z'\u002F%3E%3Cpath fill='%236f7b02' d='M147.8 115l6 32.8 5.4-17.8-51.9-17.7z'\u002F%3E%3Cpath fill='%23fff' d='M-16 134L196-16H-16z'\u002F%3E%3Cpath fill='%23272e00' d='M70.2 113.2l-7.1-20.8 4.7-1.6 7.1 20.8z'\u002F%3E%3Cpath fill='%23fff' d='M207 191L152 51l55 72z'\u002F%3E%3Cellipse fill='%239ca404' rx='1' ry='1' transform='matrix(-10.57266 -14.49873 4.04891 -2.95252 171 65)'\u002F%3E%3Cpath fill='%23170000' d='M9 194h17v10H9z'\u002F%3E%3Cpath fill='%23fff' d='M123.6-16L55.2 181.7-16 116l120.4-2z'\u002F%3E%3Cpath fill='%235b621d' d='M68.2 94.8L65.4 87l19.2-27-11.3 18z'\u002F%3E%3Cellipse fill='%23fff' cx='30' cy='232' rx='101' ry='28'\u002F%3E%3Cpath fill='%23849b90' d='M125 213l-81.7-4.9 99.8-12 15.7 2z'\u002F%3E%3Cellipse fill='%23fff' cx='138' cy='172' rx='59' ry='24'\u002F%3E%3Cpath fill='%233b432d' d='M159.6 220.6l-10.2 5.8-2-7.6 15.8-1.8z'\u002F%3E%3Cpath fill='%23a9b020' d='M147 86l1-39 17 41z'\u002F%3E%3Cpath fill='%23342312' d='M19.8 194.7l6.8 3.2 28.3 3.4-36.1 8.2z'\u002F%3E%3C\u002Fg%3E%3C\u002Fsvg%3E")},"genome-browsers.png":{filename:Y,isSvgImage:B,responsiveImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1fc8819-160.png 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1143b40-320.png 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fe8ccd21-640.png 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fdb55537-800.png 800w",images:[{path:aB,width:f,height:K},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1143b40-320.png",width:o,height:aC},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fe8ccd21-640.png",width:m,height:aD},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fdb55537-800.png",width:n,height:aE}],src:aB,width:f,height:K},responsiveWebpImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fc1517b1-160.webp 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fea2fbb6-320.webp 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fbf74273-640.webp 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff9f087b-800.webp 800w",images:[{path:aF,width:f,height:K},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fea2fbb6-320.webp",width:o,height:aC},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fbf74273-640.webp",width:m,height:aD},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff9f087b-800.webp",width:n,height:aE}],src:aF,width:f,height:K},optimizedImagePath:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff52c6a9.png",placeholderImagePath:Object("data:image\u002Fsvg+xml,%3Csvg xmlns='http:\u002F\u002Fwww.w3.org\u002F2000\u002Fsvg' viewBox='0 0 800 450'%3E%3Cfilter id='b'%3E%3CfeGaussianBlur stdDeviation='12' \u002F%3E%3C\u002Ffilter%3E%3Cpath fill='%23dbdbe4' d='M0 0h800v450H0z'\u002F%3E%3Cg filter='url(%23b)' transform='translate(1.6 1.6) scale(3.125)' fill-opacity='.5'%3E%3Cpath d='M50 113h206v11H50z'\u002F%3E%3Cellipse fill='%23000062' cx='158' cy='117' rx='111' ry='6'\u002F%3E%3Cellipse fill='%23fff' cx='25' cy='44' rx='75' ry='75'\u002F%3E%3Cellipse fill='%23fff' rx='1' ry='1' transform='matrix(-24.26125 62.7337 -39.09524 -15.11945 243.6 50)'\u002F%3E%3Cpath fill='%2361615c' d='M75 139h76v5H75z'\u002F%3E%3Cpath fill='%23000021' d='M49 17h26v3H49z'\u002F%3E%3Cpath d='M76 27l1.6 6.8L64 37l-1.6-6.8z'\u002F%3E%3Cellipse fill='%23fff' cx='18' cy='91' rx='32' ry='110'\u002F%3E%3Cellipse fill='%23000065' cx='187' cy='119' rx='140' ry='5'\u002F%3E%3Cellipse cx='117' cy='85' rx='4' ry='4'\u002F%3E%3Cpath fill='%233a3a34' d='M204 139h36v5h-36z'\u002F%3E%3Cellipse cx='199' cy='32' rx='6' ry='3'\u002F%3E%3Cpath fill='%239f9fb1' d='M65 56h182v6H65z'\u002F%3E%3Cpath fill='%23002ac1' d='M102 69h34v3h-34z'\u002F%3E%3Cellipse fill='%23fff' cx='107' cy='143' rx='13' ry='11'\u002F%3E%3Cpath fill='%23fff' d='M49 63h62v49H49z'\u002F%3E%3Cpath d='M171.3 149.7l-4-.4 1.4-13 4 .4z'\u002F%3E%3Cellipse fill='%23fff' rx='1' ry='1' transform='matrix(.0447 -8.19162 101.1909 .55215 166.2 48)'\u002F%3E%3Cellipse fill='%23fff' cx='207' rx='133' ry='13'\u002F%3E%3Cpath d='M132 28.7l1.7 2.5-5.6 4-1.8-2.4z'\u002F%3E%3C\u002Fg%3E%3C\u002Fsvg%3E")},"track-types.png":{filename:$,isSvgImage:B,responsiveImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fea8db5a-160.png 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2f69796-320.png 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F46bea7c-640.png 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1525edf-800.png 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff48bcaa-911.png 911w",images:[{path:aG,width:f,height:L},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2f69796-320.png",width:o,height:aH},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F46bea7c-640.png",width:m,height:aI},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1525edf-800.png",width:n,height:aJ},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff48bcaa-911.png",width:aK,height:aL}],src:aG,width:f,height:L},responsiveWebpImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F9a374d1-160.webp 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F68f5807-320.webp 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2664ade-640.webp 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F035cee0-800.webp 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3dceba8-911.webp 911w",images:[{path:aM,width:f,height:L},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F68f5807-320.webp",width:o,height:aH},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2664ade-640.webp",width:m,height:aI},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F035cee0-800.webp",width:n,height:aJ},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3dceba8-911.webp",width:aK,height:aL}],src:aM,width:f,height:L},optimizedImagePath:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F65ca73a.png",placeholderImagePath:Object("data:image\u002Fsvg+xml,%3Csvg xmlns='http:\u002F\u002Fwww.w3.org\u002F2000\u002Fsvg' viewBox='0 0 911 1285'%3E%3Cfilter id='b'%3E%3CfeGaussianBlur stdDeviation='12' \u002F%3E%3C\u002Ffilter%3E%3Cpath fill='%23f5f4ed' d='M0 0h908v1285H0z'\u002F%3E%3Cg filter='url(%23b)' transform='translate(2.5 2.5) scale(5.01953)' fill-opacity='.5'%3E%3Cpath fill='%23ac9c00' d='M106 144h75v6h-75z'\u002F%3E%3Cellipse fill='%23d1c655' cx='140' cy='176' rx='38' ry='7'\u002F%3E%3Cellipse fill='%23b3a100' cx='27' cy='147' rx='51' ry='3'\u002F%3E%3Cpath fill='%23ccbd1b' d='M-3 176l65 7-23-15z'\u002F%3E%3Cpath fill='%23d3c31f' d='M34 198l-33 9 61 6z'\u002F%3E%3Cpath fill='%23d1c545' d='M124 195.8l48.3 18.1-73-4 31.9-11.4z'\u002F%3E%3Cpath fill='%23afb0b7' d='M157.3 228.1l-51 14 23.3 6.9 57.9-11z'\u002F%3E%3Cellipse fill='%23fff' rx='1' ry='1' transform='matrix(-25.68752 -93.25795 46.39675 -12.7798 69.8 56)'\u002F%3E%3Cpath fill='%23c0b22a' d='M111 108h27v9h-27z'\u002F%3E%3Cpath fill='%23cdbf33' d='M5 111l27 7 2-12z'\u002F%3E%3Cpath fill='%23a99d22' d='M111 82h26v6h-26z'\u002F%3E%3Cpath fill='%23fff' d='M103.6 206.5l11 56-98.2 19-11-56z'\u002F%3E%3Cpath fill='%23bcb143' d='M9 80.3l24 .4v7l-24-.4z'\u002F%3E%3Cpath fill='%23857f43' d='M0 177h77v2H0z'\u002F%3E%3Cellipse fill='%234f4f53' rx='1' ry='1' transform='matrix(-13.95913 -.33884 .02427 -.9997 38.8 4.2)'\u002F%3E%3Cpath fill='%235c5d64' d='M118.6 130.5l-.1-1 39-2v1z'\u002F%3E%3Cellipse fill='%23fff' cx='91' cy='124' rx='16' ry='180'\u002F%3E%3Cellipse fill='%235b5840' cx='161' cy='209' rx='56' ry='1'\u002F%3E%3Cpath fill='%23ccc368' d='M60 206.9l-20.1 6.8 14.7 3.2 23.7-10.6z'\u002F%3E%3Cpath fill='%23606168' d='M123 3h35v3h-35z'\u002F%3E%3C\u002Fg%3E%3C\u002Fsvg%3E")},"under-construction.png":{filename:E,isSvgImage:B,responsiveImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2dc532f-160.png 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fe72cc86-320.png 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fbce2fe5-640.png 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff680422-800.png 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fc607bde-960.png 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F5e5e4ca-1280.png 1280w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fa2c6e33-1440.png 1440w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fb9f391e-1600.png 1600w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F9bab1b4-1920.png 1920w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2f174fc-2019.png 2019w",images:[{path:aN,width:f,height:M},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fe72cc86-320.png",width:o,height:aO},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fbce2fe5-640.png",width:m,height:aP},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff680422-800.png",width:n,height:aQ},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fc607bde-960.png",width:r,height:aR},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F5e5e4ca-1280.png",width:s,height:aS},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fa2c6e33-1440.png",width:y,height:aT},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fb9f391e-1600.png",width:z,height:aU},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F9bab1b4-1920.png",width:x,height:aV},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2f174fc-2019.png",width:aW,height:aX}],src:aN,width:f,height:M},responsiveWebpImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1afec95-160.webp 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F0bce009-320.webp 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F4ed0e34-640.webp 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd70f224-800.webp 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F90091a1-960.webp 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3c6566f-1280.webp 1280w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F96af9fe-1440.webp 1440w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F64e1327-1600.webp 1600w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F5f22ed2-1920.webp 1920w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd58cb0e-2019.webp 2019w",images:[{path:aY,width:f,height:M},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F0bce009-320.webp",width:o,height:aO},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F4ed0e34-640.webp",width:m,height:aP},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd70f224-800.webp",width:n,height:aQ},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F90091a1-960.webp",width:r,height:aR},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3c6566f-1280.webp",width:s,height:aS},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F96af9fe-1440.webp",width:y,height:aT},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F64e1327-1600.webp",width:z,height:aU},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F5f22ed2-1920.webp",width:x,height:aV},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd58cb0e-2019.webp",width:aW,height:aX}],src:aY,width:f,height:M},optimizedImagePath:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd07df6d.png",placeholderImagePath:Object("data:image\u002Fsvg+xml,%3Csvg xmlns='http:\u002F\u002Fwww.w3.org\u002F2000\u002Fsvg' viewBox='0 0 2019 2400'%3E%3Cfilter id='b'%3E%3CfeGaussianBlur stdDeviation='12' \u002F%3E%3C\u002Ffilter%3E%3Cpath fill='%238f3d00' d='M0 0h2015v2400H0z'\u002F%3E%3Cg filter='url(%23b)' transform='translate(4.7 4.7) scale(9.375)' fill-opacity='.5'%3E%3Cellipse rx='1' ry='1' transform='matrix(82.86672 -154.26602 39.10804 21.00758 24.5 46.5)'\u002F%3E%3Cellipse rx='1' ry='1' transform='matrix(-84.17916 -161.3501 36.61345 -19.10187 184 38.5)'\u002F%3E%3Cellipse fill='%23ff8e00' cx='107' cy='200' rx='106' ry='106'\u002F%3E%3Cpath fill='%23ff2c00' d='M108-11l122 207-235-14z'\u002F%3E%3Cellipse fill='%238dbc00' rx='1' ry='1' transform='matrix(-42.7985 5.33083 -6.1152 -49.09574 109.4 118.8)'\u002F%3E%3Cellipse rx='1' ry='1' transform='matrix(-29.332 13.26803 -63.51772 -140.42045 190.3 61.3)'\u002F%3E%3Cpath d='M142 111l-50 49 18-77z'\u002F%3E%3Cellipse rx='1' ry='1' transform='matrix(-56.10465 106.86361 -34.70465 -18.22035 22.5 57)'\u002F%3E%3Cellipse fill='%23ffe300' cx='106' cy='76' rx='19' ry='37'\u002F%3E%3Cellipse fill='%23ff0' cx='38' cy='210' rx='37' ry='16'\u002F%3E%3Cpath d='M208 188.4v7l-100-1.8v-7z'\u002F%3E%3Cellipse fill='%23fffc00' rx='1' ry='1' transform='rotate(-23.5 412.8 -295.2) scale(16.50896 31.12544)'\u002F%3E%3Cpath fill='%23ff0' d='M146 194h65v33h-65z'\u002F%3E%3Cellipse fill='red' cx='63' cy='177' rx='214' ry='12'\u002F%3E%3Cellipse cx='86' cy='191' rx='148' ry='4'\u002F%3E%3Cpath fill='%23ff0' d='M75 164l-44 3 58-98z'\u002F%3E%3Cpath d='M219.5 253v4h-223v-4z'\u002F%3E%3Cellipse fill='red' rx='1' ry='1' transform='matrix(-11.08132 -6.21855 64.915 -115.67715 53.4 105.6)'\u002F%3E%3Cellipse rx='1' ry='1' transform='matrix(-61.96942 -78.65617 17.25085 -13.59112 175.3 55.2)'\u002F%3E%3Cellipse rx='1' ry='1' transform='rotate(-46.1 65.2 3.7) scale(116.75051 41.2497)'\u002F%3E%3C\u002Fg%3E%3C\u002Fsvg%3E")},"track-collections.png":{filename:al,isSvgImage:B,responsiveImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F81ad884-160.png 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fc1d756d-320.png 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1943e11-640.png 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F84acdbe-800.png 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fbc3d18b-960.png 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3d88da6-1200.png 1200w",images:[{path:aZ,width:f,height:D},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fc1d756d-320.png",width:o,height:C},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1943e11-640.png",width:m,height:F},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F84acdbe-800.png",width:n,height:N},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fbc3d18b-960.png",width:r,height:m},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3d88da6-1200.png",width:a_,height:n}],src:aZ,width:f,height:D},responsiveWebpImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F0bc2586-160.webp 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F74a7104-320.webp 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F22230c1-640.webp 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fa0d266b-800.webp 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fcc77733-960.webp 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F4233362-1200.webp 1200w",images:[{path:a$,width:f,height:D},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F74a7104-320.webp",width:o,height:C},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F22230c1-640.webp",width:m,height:F},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fa0d266b-800.webp",width:n,height:N},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fcc77733-960.webp",width:r,height:m},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F4233362-1200.webp",width:a_,height:n}],src:a$,width:f,height:D},optimizedImagePath:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F359a83b.png",placeholderImagePath:Object("data:image\u002Fsvg+xml,%3Csvg xmlns='http:\u002F\u002Fwww.w3.org\u002F2000\u002Fsvg' viewBox='0 0 1200 800'%3E%3Cfilter id='b'%3E%3CfeGaussianBlur stdDeviation='12' \u002F%3E%3C\u002Ffilter%3E%3Cpath fill='%23f0ede9' d='M0 0h1200v796H0z'\u002F%3E%3Cg filter='url(%23b)' transform='translate(2.3 2.3) scale(4.6875)' fill-opacity='.5'%3E%3Cellipse fill='%23c49d7f' rx='1' ry='1' transform='matrix(-4.70402 28.11651 -68.25282 -11.419 101.4 107.8)'\u002F%3E%3Cpath fill='%23ffa128' d='M51 49h19v83H51z'\u002F%3E%3Cellipse fill='%23fff' cx='173' cy='52' rx='104' ry='43'\u002F%3E%3Cpath fill='%237fb3f6' d='M28 41h21v92H28z'\u002F%3E%3Cpath fill='%2366fd72' d='M75 131.5l17.6 4.3 1.1-49-20.9-11z'\u002F%3E%3Cpath fill='%23bdb5ae' d='M140 111h110v24H140z'\u002F%3E%3Cellipse fill='%23fff' rx='1' ry='1' transform='matrix(8.58635 -22.7231 224.05783 84.6644 145.5 54.8)'\u002F%3E%3Cellipse fill='%23fff' cx='133' cy='142' rx='254' ry='9'\u002F%3E%3Cellipse fill='%23999c9f' cx='132' cy='7' rx='85' ry='3'\u002F%3E%3Cpath fill='%23ff7380' d='M98 83h18v50H98z'\u002F%3E%3Cellipse fill='%23a2a1a0' cx='162' cy='152' rx='163' ry='2'\u002F%3E%3Cpath fill='%23cdc4ff' d='M115 91h26v44h-26z'\u002F%3E%3Cellipse fill='%23fff' rx='1' ry='1' transform='rotate(-86.5 55.6 41.4) scale(95.11682 17.34756)'\u002F%3E%3Cpath fill='%23b77876' d='M141.5 132l2-39 17 1-2 39z'\u002F%3E%3Cellipse fill='%23fff' cx='229' cy='52' rx='56' ry='56'\u002F%3E%3Cpath fill='%23e0e653' d='M208 113h19v21h-19z'\u002F%3E%3Cpath fill='%236ee3fd' d='M232.2 110.8l18 .4-.4 24-18-.4z'\u002F%3E%3Cpath fill='%23ffc1ef' d='M187 102l-1 29-27-1 1-29z'\u002F%3E%3Cellipse fill='%23fff' cx='137' cy='169' rx='255' ry='9'\u002F%3E%3Cpath fill='%23abb3b3' d='M191 156.7l-2.9-7 12.8 11.4-133.3-2.4z'\u002F%3E%3C\u002Fg%3E%3C\u002Fsvg%3E")},"identifiers.jpeg":{filename:ap,isSvgImage:B,responsiveImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff774ed9-160.jpeg 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2a3600a-320.jpeg 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F0ddb65c-640.jpeg 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F17352fa-800.jpeg 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fbaf05a3-960.jpeg 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F507323e-1280.jpeg 1280w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fb0a2609-1440.jpeg 1440w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1a5c99e-1600.jpeg 1600w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F5558c15-1920.jpeg 1920w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1b0c19e-2560.jpeg 2560w",images:[{path:ba,width:f,height:D},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2a3600a-320.jpeg",width:o,height:C},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F0ddb65c-640.jpeg",width:m,height:F},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F17352fa-800.jpeg",width:n,height:N},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fbaf05a3-960.jpeg",width:r,height:m},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F507323e-1280.jpeg",width:s,height:J},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fb0a2609-1440.jpeg",width:y,height:r},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1a5c99e-1600.jpeg",width:z,height:bb},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F5558c15-1920.jpeg",width:x,height:s},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1b0c19e-2560.jpeg",width:G,height:bc}],src:ba,width:f,height:D},responsiveWebpImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Faaa545b-160.webp 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fa7d75e5-320.webp 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3f52f1f-640.webp 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F13a583d-800.webp 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F26b7c66-960.webp 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fbbe061f-1280.webp 1280w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F433ca74-1440.webp 1440w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F24088e1-1600.webp 1600w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F5169556-1920.webp 1920w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F49457ef-2560.webp 2560w",images:[{path:bd,width:f,height:D},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fa7d75e5-320.webp",width:o,height:C},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3f52f1f-640.webp",width:m,height:F},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F13a583d-800.webp",width:n,height:N},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F26b7c66-960.webp",width:r,height:m},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fbbe061f-1280.webp",width:s,height:J},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F433ca74-1440.webp",width:y,height:r},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F24088e1-1600.webp",width:z,height:bb},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F5169556-1920.webp",width:x,height:s},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F49457ef-2560.webp",width:G,height:bc}],src:bd,width:f,height:D},optimizedImagePath:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fe9db64a.jpeg",placeholderImagePath:Object("data:image\u002Fsvg+xml,%3Csvg xmlns='http:\u002F\u002Fwww.w3.org\u002F2000\u002Fsvg' viewBox='0 0 3376 2250'%3E%3Cfilter id='b'%3E%3CfeGaussianBlur stdDeviation='12' \u002F%3E%3C\u002Ffilter%3E%3Cpath fill='%237db1d4' d='M0 0h3376v2241H0z'\u002F%3E%3Cg filter='url(%23b)' transform='translate(6.6 6.6) scale(13.1875)' fill-opacity='.5'%3E%3Cpath fill='%23882e00' d='M48.7 89.9l2.8-39 73.8 5.2-2.8 39z'\u002F%3E%3Cpath fill='%23000078' d='M231.5 101v24h-99v-24z'\u002F%3E%3Cellipse fill='%23009dff' rx='1' ry='1' transform='matrix(96.78296 64.05924 -16.73649 25.28608 244 14)'\u002F%3E%3Cellipse fill='%23bae5f7' rx='1' ry='1' transform='rotate(24.9 -266.8 139.4) scale(71.18835 43.03235)'\u002F%3E%3Cellipse fill='%23bde2fa' cx='189' cy='61' rx='56' ry='39'\u002F%3E%3Cellipse fill='%23ffdf00' rx='1' ry='1' transform='matrix(.52991 -5.60588 42.22726 3.99165 88.7 89)'\u002F%3E%3Cellipse fill='%2300b5ef' rx='1' ry='1' transform='rotate(-96.6 108.2 27.8) scale(5.82397 37.57846)'\u002F%3E%3Cellipse fill='%23005f00' cx='87' cy='74' rx='38' ry='5'\u002F%3E%3Cpath fill='%23fff8d9' d='M212 30L84 28l49 22z'\u002F%3E%3Cpath fill='%23009efd' d='M231.5 60.3L204.2-15l42.3-15.4L273.8 45z'\u002F%3E%3Cellipse fill='%23cc002b' cx='91' cy='60' rx='38' ry='5'\u002F%3E%3Cellipse fill='%2315a9f8' cx='163' cy='17' rx='59' ry='11'\u002F%3E%3Cpath fill='%238d009f' d='M132 114h81v10h-81z'\u002F%3E%3Cellipse fill='%23000a83' cx='185' cy='105' rx='54' ry='6'\u002F%3E%3Cpath fill='%23005774' d='M46 35l83 12-43 6z'\u002F%3E%3Cellipse fill='%232abdff' rx='1' ry='1' transform='rotate(72.1 43 230) scale(30.80432 19.93232)'\u002F%3E%3Cpath fill='%2300422a' d='M130.2 160.2l6.6 24.8-4.4-43.1 17.3 15.7z'\u002F%3E%3Cellipse fill='%2382c9ff' cx='217' cy='155' rx='72' ry='32'\u002F%3E%3Cpath fill='%2344a5eb' d='M172 38h68v21h-68z'\u002F%3E%3Cpath fill='%23bed8e7' d='M124.2 96.3l8.5 30.8L-16 145.3l93-52z'\u002F%3E%3C\u002Fg%3E%3C\u002Fsvg%3E")},"ontologies.png":{filename:as,isSvgImage:B,responsiveImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1ff4490-160.png 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fb1b7839-320.png 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3a2d08a-640.png 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F96019d5-800.png 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fb07c0e9-960.png 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F45a6864-961.png 961w",images:[{path:be,width:f,height:O},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fb1b7839-320.png",width:o,height:bf},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3a2d08a-640.png",width:m,height:bg},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F96019d5-800.png",width:n,height:bh},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fb07c0e9-960.png",width:r,height:bi},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F45a6864-961.png",width:bj,height:bk}],src:be,width:f,height:O},responsiveWebpImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F87e8435-160.webp 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fade0757-320.webp 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fefb1e63-640.webp 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F8efc590-800.webp 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fad32acf-960.webp 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F6576c18-961.webp 961w",images:[{path:bl,width:f,height:O},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fade0757-320.webp",width:o,height:bf},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fefb1e63-640.webp",width:m,height:bg},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F8efc590-800.webp",width:n,height:bh},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fad32acf-960.webp",width:r,height:bi},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F6576c18-961.webp",width:bj,height:bk}],src:bl,width:f,height:O},optimizedImagePath:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff2d7581.png",placeholderImagePath:Object("data:image\u002Fsvg+xml,%3Csvg xmlns='http:\u002F\u002Fwww.w3.org\u002F2000\u002Fsvg' viewBox='0 0 961 486'%3E%3Cfilter id='b'%3E%3CfeGaussianBlur stdDeviation='12' \u002F%3E%3C\u002Ffilter%3E%3Cpath fill='%23eff4f8' d='M0 0h961v484H0z'\u002F%3E%3Cg filter='url(%23b)' transform='translate(1.9 1.9) scale(3.7539)' fill-opacity='.5'%3E%3Cellipse fill='%23176094' cx='95' cy='115' rx='149' ry='5'\u002F%3E%3Cpath fill='%2300025a' d='M196 51h47v8h-47z'\u002F%3E%3Cpath fill='%23001265' d='M77 51h20v9H77z'\u002F%3E%3Cpath fill='%23002774' d='M207 11.2l26-.4v8l-26 .4z'\u002F%3E%3Cellipse fill='%23fff' rx='1' ry='1' transform='matrix(-6.61798 -59.94487 19.87547 -2.19427 184.2 93.6)'\u002F%3E%3Cellipse fill='%23fff' cx='135' cy='61' rx='13' ry='95'\u002F%3E%3Cpath fill='%23002b75' d='M207 111h26v8h-26z'\u002F%3E%3Cellipse fill='%23063a63' cx='70' cy='115' rx='63' ry='2'\u002F%3E%3Cpath fill='%23001266' d='M83 12h8v9h-8z'\u002F%3E%3Cpath fill='%23fff' d='M24.4 84.6l9 40.7 59.4-55.7L-16-16z'\u002F%3E%3Cpath fill='%23fff' d='M97.4 112.1L240.4-16l-166 84.9 20 75.1z'\u002F%3E%3Cpath fill='%23fff' d='M148.5 108.8L-16 109.9l28 25.6L-16-8z'\u002F%3E%3Cellipse fill='%23003b6a' cx='17' cy='113' rx='11' ry='4'\u002F%3E%3Cpath fill='%23004386' d='M148 111h19v8h-19z'\u002F%3E%3Cellipse fill='%23fff' cx='253' cy='128' rx='20' ry='92'\u002F%3E%3Cpath fill='%23fff' d='M230 0h26v51h-26z'\u002F%3E%3Cpath fill='%23005191' d='M43 111h43v7H43z'\u002F%3E%3Cpath fill='%23006f9e' d='M201 52h39v8h-39z'\u002F%3E%3Cpath fill='%23fff' d='M167 63h37v66h-37z'\u002F%3E%3Cpath fill='%23fff' d='M219-5l-86 137L84-16z'\u002F%3E%3C\u002Fg%3E%3C\u002Fsvg%3E")},"validation.png":{filename:at,isSvgImage:B,responsiveImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F286564e-160.png 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F282aac8-320.png 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F820c977-640.png 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3e809ea-800.png 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ffd8f0b6-960.png 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fc14bd10-1280.png 1280w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F476dec3-1440.png 1440w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F095fd0c-1600.png 1600w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F66c26ad-1920.png 1920w",images:[{path:bm,width:f,height:f},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F282aac8-320.png",width:o,height:o},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F820c977-640.png",width:m,height:m},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F3e809ea-800.png",width:n,height:n},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ffd8f0b6-960.png",width:r,height:r},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fc14bd10-1280.png",width:s,height:s},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F476dec3-1440.png",width:y,height:y},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F095fd0c-1600.png",width:z,height:z},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F66c26ad-1920.png",width:x,height:x}],src:bm,width:f,height:f},responsiveWebpImage:{srcSet:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fb00ceec-160.webp 160w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F5a23aed-320.webp 320w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F9fb791a-640.webp 640w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fcb5c599-800.webp 800w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F268f8e5-960.webp 960w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F0c19b95-1280.webp 1280w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F0563c6c-1440.webp 1440w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd643c9f-1600.webp 1600w,\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ffe7c21a-1920.webp 1920w",images:[{path:bn,width:f,height:f},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F5a23aed-320.webp",width:o,height:o},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F9fb791a-640.webp",width:m,height:m},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fcb5c599-800.webp",width:n,height:n},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F268f8e5-960.webp",width:r,height:r},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F0c19b95-1280.webp",width:s,height:s},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F0563c6c-1440.webp",width:y,height:y},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd643c9f-1600.webp",width:z,height:z},{path:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ffe7c21a-1920.webp",width:x,height:x}],src:bn,width:f,height:f},optimizedImagePath:"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2dcb7e8.png",placeholderImagePath:Object("data:image\u002Fsvg+xml,%3Csvg xmlns='http:\u002F\u002Fwww.w3.org\u002F2000\u002Fsvg' viewBox='0 0 1920 1920'%3E%3Cfilter id='b'%3E%3CfeGaussianBlur stdDeviation='12' \u002F%3E%3C\u002Ffilter%3E%3Cpath fill='%23605d5a' d='M0 0h1920v1920H0z'\u002F%3E%3Cg filter='url(%23b)' transform='matrix(7.5 0 0 7.5 3.8 3.8)' fill-opacity='.5'%3E%3Cellipse fill='%23fff' rx='1' ry='1' transform='rotate(115.1 19.8 103.8) scale(75.66773 100.38036)'\u002F%3E%3Cellipse rx='1' ry='1' transform='matrix(-173.8813 -138.31143 32.51714 -40.87965 210.8 46.8)'\u002F%3E%3Cellipse rx='1' ry='1' transform='matrix(41.66765 156.2106 -35.08536 9.35868 16.8 186.9)'\u002F%3E%3Cellipse rx='1' ry='1' transform='matrix(207.41999 -104.77537 18.7209 37.0611 204.3 232.3)'\u002F%3E%3Cellipse rx='1' ry='1' transform='matrix(6.96507 34.2344 -222.94738 45.35916 73.1 22.5)'\u002F%3E%3Cellipse fill='%23fff' cx='118' cy='138' rx='82' ry='82'\u002F%3E%3Cellipse fill='%23fff' rx='1' ry='1' transform='matrix(-5.23481 29.47733 -41.52824 -7.3749 200.1 148.2)'\u002F%3E%3Cellipse rx='1' ry='1' transform='matrix(-20.11531 -35.85963 165.42302 -92.79336 213.1 226.8)'\u002F%3E%3Cellipse rx='1' ry='1' transform='rotate(133.7 95 79.4) scale(41.82432 130.96081)'\u002F%3E%3Cellipse cy='199' rx='63' ry='63'\u002F%3E%3Cellipse rx='1' ry='1' transform='matrix(152.75306 -42.46164 9.38297 33.75465 55 24.2)'\u002F%3E%3Cpath fill='%23effaff' d='M84.4 238.3l-18.3-34.5 41.5-22 18.3 34.4z'\u002F%3E%3Cpath fill='%23fff' d='M19 70h95v41H19z'\u002F%3E%3Cpath fill='%23fff' d='M134.1 37.8L62.4 59.6l15-2.1 164.4 112z'\u002F%3E%3Cpath fill='%2300408e' d='M129.3 95.9L112 83.7l47.8 96-9.5-43.5z'\u002F%3E%3Cellipse fill='%23c56111' rx='1' ry='1' transform='matrix(-2.9693 -5.26965 81.50765 -45.9272 134.7 217.9)'\u002F%3E%3Cpath d='M-10 34l1 221 92 16z'\u002F%3E%3Cellipse rx='1' ry='1' transform='matrix(22.83857 -50.81812 113.44708 50.98514 222.3 36.3)'\u002F%3E%3Cellipse fill='%23a19e9d' rx='1' ry='1' transform='rotate(-19.3 497 -119.4) scale(28.01749 75.14303)'\u002F%3E%3Cpath fill='%23fff' d='M132.8 212.9l-68-113.2 34.4-20.6 68 113.2z'\u002F%3E%3C\u002Fg%3E%3C\u002Fsvg%3E")}}}],fetch:{},mutations:[]}}("text","element","\n","p","a",160,"nofollow","noopener","noreferrer","_blank","strong","2022-04-05T12:01:21.164Z",640,800,320,"li","em",960,1280,"TBA","root","\u002Ftopics",".md",1920,1440,1600,".",false,213,107,"under-construction.png",427,2560,"lengths"," and ",853,90,226,190,533,81,"1","\n\n\n","gaps","interconnections","nuxt-link","genomic-tracks.jpeg","Genomic tracks","ui-quote-text","ul","genome-browsers.png","ui-markdown-table","'Various genome browsers deployed as web services with possibly domain-restricted track ' +\n'databases and\u002For available as installable software.'","track-types.png","et al."," ","https:\u002F\u002Fdoi.org\u002F10.1186\u002F1471-2105-12-494","(2011)","track types",", ","values","ol","Identifiers","s09-ontologies","Ontologies","track-collections.png","https:\u002F\u002Fwww","https:\u002F\u002Frepositoryfinder.datacite.org","http:\u002F\u002Fgenome.ucsc.edu\u002Fcgi-bin\u002FhgHubConnect","identifiers.jpeg","http:\u002F\u002Fidentifiers.org\u002F","Identifiers.org","ontologies.png","validation.png","https:\u002F\u002Fwww.ga4gh.org\u002Fgenomic-data-toolkit\u002F","\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fd56a02a-160.jpeg",1067,1707,2133,3413,"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F78c3436-160.webp","\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1fc8819-160.png",180,360,450,"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fc1517b1-160.webp","\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fea8db5a-160.png",451,903,1128,911,1285,"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F9a374d1-160.webp","\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F2dc532f-160.png",380,761,951,1141,1522,1712,1902,2282,2019,2400,"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1afec95-160.webp","\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F81ad884-160.png",1200,"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F0bc2586-160.webp","\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Ff774ed9-160.jpeg",1066,1706,"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Faaa545b-160.webp","\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F1ff4490-160.png",162,324,405,485,961,486,"\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F87e8435-160.webp","\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002F286564e-160.png","\u002Ffairtracks.net\u002F_nuxt\u002Fimg\u002Fb00ceec-160.webp")));